{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "7cde5eb1",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[\"JISCMail - CCPEM Archives Print Hi all, In RELION, is it possible to generate a Fourier space diffraction \\npattern of the reconstructed volume Thanks in advance for any advice. Abhi \\n######################################################################## To unsubscribe \\nfrom the CCPEM list, click the  following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message was issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing list hosted by www.jiscmail.ac.uk, terms & conditions \\nare available at https://www.jiscmail.ac.uk/pol icyandsecurity/ JISCMail - CCPEM Archives Print Hi all, Are \\nyou ready to start a new career within the vibrant community of structural scientists in NYC? Energized \\nby developing new workflows by being a part of one of the leading structural biology centers  in the \\nworld? Looking forward to making cryoEM not only part of your profession but research passion? We \\nhave openings for: Staff microscopist RNA Structural Biology postdoc Proteomics postdoc or Staff \\nscientist 3D -EM Image Analyst ...and more! See all ou r opportunities at: \\nhttps://semc.nysbc.org/job -opportunities/ If you have questions let me know if I can help. Also, you'll \\nfind us July 7 -11 at the ACA annual meeting in Baltimore, MD at booth #105, July 16 -21 at the GRC in \\nVolume EM in Ventura, CA; and J uly 23 -27 at the M&M annual meeting in Minneapolis, MN at booth \\n#1039. Best, Ed --- Edward T. Eng, Ph.D. Simons Electron Microscopy Center New York Structural Biology \\nCenter 89 Convent Ave, NY , NY 10027 semc.nysbc.org | (212)939 -0660 x344 \\n################# ####################################################### To unsubscribe \\nfrom the CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message was issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing list hosted by www.jiscmail.ac.uk, terms & conditions \\nare available at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print Hi all, In \\nRELION, is it possible to generate a Fourier space diffraction pattern of the reconstructed volu me Thanks \\nin advance for any advice. Abhi \\n######################################################################## To unsubscribe \\nfrom the CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message wa s issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing list hosted by www.jiscmail.ac.uk, terms & conditions \\nare available at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print Hello \\nEveryone, Please join our virtual event to  learn how the combination of the 100kV Tundra Cryo -TEM with \\nthe new Falcon C Direct Electron Detector not only improves resolution, but also allows for structure \\ndetermination of small and challenging proteins. Attend the webinar, access helpful resources , schedule \\na demo, and connect with experts to learn more. Use the link below register for the event: | July 13 | \\n11:00 a.m. EDT | 8:00 a.m. PDT | 5:00 p.m. CEST Tundra with Falcon C registration  \\n<http://thermofisher.com/content/lifetech/global/en/home/ab out-us/events/industrial/tundra -cryo -te\\nm.html> Best regards, Jeff \\n____________________________________________________________________ Jeffrey Lengyel, Ph.D. \\nDirector, Product Marketing, Cryo -EM, Electron Microscopy Life Sciences Materials & Structural Ana lysis \\nThermo Fisher Scientific 5350 NE Dawson Creek Drive | Hillsboro, OR 97124 Phone 503 560 9184 [log in \\nto unmask]<mailto:[log in to unmask]> [cid:image001.jpg@01D9AB28.D772B240] If you are interested in \\nhearing more about events and webinars such as th is one hosted by the electron microscopy team, \\nsubscribe<https://www.thermofisher.com/us/en/home/global/forms/industrial/life -science -research -be\\n-the-first-to-know.html> to our email list and you will be automated notified of upcoming events. Hope \\nto see y ou there! \", \"######################################################################## To unsubscribe \\nfrom the CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message was issued to \\nmembers of www.jiscm ail.ac.uk/CCPEM, a mailing list hosted by www.jiscmail.ac.uk, terms & conditions \\nare available at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print Hi Forhad, \\nIf you notice dewetting of your sample on the grid (i.e. puddles in t he center of your squares) when you \\nprepare your grids after glow discharging, you likely have residual plastic contamination on your \\nQuantifoil grids. We received a large batch of UltrAuFoils that have to be washed with acetone or ethyl \\nacetate before use , as they're all contaminated. Be wary of plasma cleaning your UltrAuFoil grids too \\nharshly, as you'll start to see bits of gold in your cryoEM images. best of luck, -gabe > On Jun 29, 2023, at \\n12:19 PM, Lifei <[log in to unmask]> wrote: >  > Hi, Forhad, >   > the commercially purchased grids can \\nbe used directly since the manufacturing processes minimized residual contamination. Anyway, we need \\nto use the plasma cleaning (glow discharging) to remove organic contamination and increase its \\nhydrophilicity, sin ce the support films are usually hydrophobic, the hydrophobic film repels liquid, \\ncausing a drop of solution to accumulate in the center of the grid. Increasing its hydrophilicity allows for \\nbetter dispersion of protein solution liquid across foil, after v itrification, more chance to obtain a very \\nthin and uniform layer of vitreous ice, so the wetting level has an influence on the control particle \\ndispersion homogeneously over the grid. >  > In the plasma cleaning system, electrons are accelerated \\nin the el ectric field to gain enough high energy and then collide with molecules from a low -pressure gas \\ninside the sample chamber. When the voltage exceeds a striking range, the gas molecules' covalent \\nbonds break up into charged particles such as ions and radical s with glowing light, these chemically \\nreactive gases react with hydrocarbon mono -layers to form gaseous products that are swept away by the \\ncontinuous gas flow (Drulyte et al., 2018), so it can remove hydrocarbon accumulation (residual organic \\ncontaminati on and depositions) on the grid. This plasma reacts chemically with the surface, generating \\nhydrophilic functional groups, and thereby reducing hydrophobicity or increasing the hydrophilicity of \\nthe grids temporarily. >  > Gold is a highly conductive, non -oxidizing, radiation -resistance, chemically \\nstable, and biocompatible material (Russo and Passmore, 2015), but is also not hydrophilic, it resists \\nchemical reactions with the typical residual gases in a vacuum chamber, so we can apply more \\nextensively plas ma clean to make it more hydrophilic, there is no risk of damaging UltrAuFoil films by \\nlonger plasma -cleaning. >  > The hydrophilicity can be measured by the air water film contact angle, the \\ncontact area between the grid surface and sample droplet should be maintained with a low contact \\nangle (such as below 40 degrees), more info can find from wiki/Plasma_cleaning, Russo and Passmore's \\npaper. >  > Best > Lifei >  >  >> On 26.06.23 03:00, Forhad Karim Saikot wrote: >> Hi all,  >>  >> \\nApologies, but I have a  quick question that suits the thread nicely. Should we wash UltraAuFoil grids with \\nEthyl Acetate or similar solvent, the way we do for Quantifoil grids?  >>  >> Thanks in \\nadvance.  >>  >>  >> Kind regards, >>  >>  >>  >> Forhad Karim Saikot >>  >> PhD Stu dent >>  >> \\nBostjan Kobe Lab >  > \\n######################################################################## >  > To \\nunsubscribe from the CCPEM list, click the following link: > \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 >  > This messag e was issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing list hosted by www.jiscmail.ac.uk, terms & conditions \\nare available at https://www.jiscmail.ac.uk/policyandsecurity/ \\n######################################################################## To unsubscribe \\nfrom the CCPEM list, click the following link: \", \"https://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message was issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing list hosted by www.jiscmail.ac.uk, terms & conditions \\nare a vailable at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print Hi, \\nForhad, the commercially purchased grids can be used directly since the manufacturing processes \\nminimized residual contamination. Anyway, we need to use the plasm a cleaning (glow discharging) to \\nremove organic contamination and increase its hydrophilicity, since the support films are usually \\nhydrophobic, the hydrophobic film repels liquid, causing a drop of solution to accumulate in the center \\nof the grid. Increasi ng its hydrophilicity allows for better dispersion of protein solution liquid across foil, \\nafter vitrification, more chance to obtain a very thin and uniform layer of vitreous ice, so the wetting \\nlevel has an influence on the control particle dispersion ho mogeneously over the grid. In the plasma \\ncleaning system, electrons are accelerated in the electric field to gain enough high energy and then \\ncollide with molecules from a low -pressure gas inside the sample chamber. When the voltage exceeds a \\nstriking rang e, the gas molecules' covalent bonds break up into charged particles such as ions and \\nradicals with glowing light, these chemically reactive gases react with hydrocarbon mono -layers to form \\ngaseous products that are swept away by the continuous gas flow (D rulyte et al., 2018), so it can remove \\nhydrocarbon accumulation (residual organic contamination and depositions) on the grid. This plasma \\nreacts chemically with the surface, generating hydrophilic functional groups, and thereby reducing \\nhydrophobicity or i ncreasing the hydrophilicity of the grids temporarily. Gold is a highly conductive, \\nnon-oxidizing, radiation -resistance, chemically stable, and biocompatible material (Russo and Passmore, \\n2015), but is also not hydrophilic, it resists chemical reactions wi th the typical residual gases in a vacuum \\nchamber, so we can apply more extensively plasma clean to make it more hydrophilic, there is no risk of \\ndamaging UltrAuFoil films by longer plasma -cleaning. The hydrophilicity can be measured by the air \\nwater film contact angle, the contact area between the grid surface and sample droplet should be \\nmaintained with a low contact angle (such as below 40 degrees), more info can find from \\nwiki/Plasma_cleaning, Russo and Passmore's paper. Best Lifei >On 26.06.23 03:00, F orhad Karim Saikot \\nwrote: > Hi all,  > > Apologies, but I have a quick question that suits the thread nicely. Should we wash \\nUltraAuFoil grids with Ethyl Acetate or similar solvent, the way we do for Quantifoil grids?  > > Thanks in \\nadvance.  > > > Kind re gards, > >   > > Forhad Karim Saikot > > PhD Student > > Bostjan Kobe Lab \\n######################################################################## To unsubscribe \\nfrom the CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe? SUBED1=CCPEM&A=1 This message was issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing list hosted by www.jiscmail.ac.uk, terms & conditions \\nare available at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print Hi \\nTakanori, tha nks so much for the clarification. Very helpful indeed! My apologies  for dropping the CC \\nto CCPEM. Best Isai <mailto:[log in to unmask]> On 29/06/2023 16:16, Takanori Nakane wrote: > Hi, > > \\nPlease don't drop CC to CCPEM. > > > If possible, could you kind ly elaborate > > a little on the problem \\nwith saturating A40 cards with RELION. does  > this > > have to do with the box size, for ex? > > GPU is a \\nmassively parallel processor. Ideally, the processing > performance should increase linearly with the \\nnumber  of (GPU) cores. > To achieve this, all cores have to perform processing tasks > simultaneously \\nand nobody should remain idle. > > Unfortunately, RELION's implementation do not have such a high > \\nlevel of parallelism to utilize all cores in the latest GPUs . > For example, multiple tasks cannot \\naccumulate into a single > voxel (called a race condition) during back -projection. > While one task writes, \\nall others have to wait. > > One way to mitigate this is to run multiple RELION processes > on a GPU. But \\nthis multiplies GPU memory requirements. > This is OK for A40, which has 48 GB of RAM, but the \", \"memory > bandwidth then becomes limiting. Also note that more > MPI processes need more CPU \\npowers and coordination overheads > among MPI processes start to grow. >  > In my opinion, A5000 is \\nwell balanced among Tesla and Quadro cards. > I would run one or two MPI processes on a A5000. > > Of \\ncourse the sweet spot depends on the box size, > resolution etc. If your group always work on huge \\nvirus > datasets, it is a di fferent story. But for general, diverse > workloads expected in a shared facility, \\nA5000 should be > adequate. > > That being said, I haven't tested all cards. You should > run benchmarks \\nof realistic workloads (not our old Ribosome Class3D > benchmark). M ost HPC vendors let you test -drive \\ntheir system before > placing orders. > > Best regards, > > Takanori Nakane > > On 6/29/23 23:00, \\nEaazhisai KANDIAH wrote: >> Dear Takanori, >> >> Thanks very much for your input. If possible, could \\nyou kindly  >> elabora te a little on the problem with saturating A40 cards with  >> RELION. does this \\nhave to do with the box size, for ex? A5000 cards  >> are in the menu as well. Some colleagues also \\nrecommended A6000  >> cards, but i guess the difference might be little betw een these two. >> >> A \\nload of thanks again, >> >> Best >> >> Isai >> >> <mailto:[log in to unmask]> >> >> On 29/06/2023 04:00, \\nTakanori Nakane wrote: >>> Hi, >>> >>> > Do RELION and CRYOSPARC take advantage on the capabilities \\noffered  >>> by NVLink. >>> >>> RELION don't use NVLink. >>> >>> > Lastly, are there any \\nalternative >>> > GPU cards currently available that offer similar functionality to the >>> > A40 but at a \\nmore affordable price? >>> >>> Assuming you want to avoid GeForce cards, my recommendati on for >>> \\nRELION (I cannot comment on other programs) is NVIDIA A5000. >>> >>> A40 is very hard to saturate \\nwith RELION, both in terms >>> of single precision FLOPS and GPU memory. >>> You can buy two A5000s \\nfor the price of one A40 and this should give > >> better aggregate throughput. >>> >>> FYI, we are \\nbuilding a GPU cluster for SPA. >>> One node will have 4 x A5000s, 512 GB of RAM and >>> 64 CPU cores \\n(128 threads). >>> One node will host two jobs (2 GPUs each, divided by cgroups) >>> or one job (4 GPU s; \\nworse total throughput but lower run -time). >>> >>> Best regards, >>> >>> Takanori Nakane >>> >>> On \\n2023/06/29 5:29, Eaazhisai KANDIAH wrote: >>>> Dear all, >>>> >>>> I would appreciate suggestions or \\nreferences to benchmark studies  >>>> regarding the  utilization of A40 GPU cards in a workstation \\nto  >>>> execute RELION and CRYOSPARC. Given that A40 cards are typically  >>>> recommended for \\nservers due to their passive cooling, I am  >>>> concerned about potential heating problems if they are \\nemployed in  >>>> a workstation. Any extra fittings are needed in the chassis to  >>>> adopt these in a \\nworkstation? Additionally, for a workstation  >>>> equipped with 4 or 8 GPUs, would it be \\nadvantageous to opt for  >>>> NVLink? Do RELION and CRYOSPARC take adva ntage on the \\ncapabilities  >>>> offered by NVLink. Lastly, are there any alternative GPU cards  >>>> currently \\navailable that offer similar functionality to the A40 but  >>>> at a more affordable price? >>>> >>>> \\nThanks so much in advance for your time and  help, >>>> >>>> Best >>>> >>>> Isai >>>> >>>> \\n--  >>>> >>>> *Dr. Eaazhisai Kandiah* >>>> >>>> CM01 beamline >>>> >>>> \\nhttps://www.esrf.fr/CM01 >>>> >>>> 71 Avenue des Martyrs, 38000 Grenoble >>>> Telephone: +33 476 \\n882691 >>>> Email: [log in to unmask] <m ailto:[log in to unmask]> >>>> >>>> >>>> \\n------------------------------------------------------------------------   >>>> >>>> >>>> To unsubscribe from the \\nCCPEM list, click the following link: >>>> \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM &A=1  >>>> \\n<https://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1> >>>> \\n######################################################################## To unsubscribe \\nfrom the CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message was issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing list hosted by www.jiscmail.ac.uk, terms & conditions \", \"are available at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print Hi,  > \\nAfter CTF refining, the particles were not aligning anymore upon 3D  refinement. Therefore I tried \\n3D-refining after each CTF step and I  figured out the problem was the 4th order aberrations \\nparamenter: when  set to No, refinement was then working correctl y (I didn t use trefoil  estimation). \\nJust to make sure, didn't you enable B factor refinement? That should be OFF unless the dataset is \\nREALLY good. Do phase error plots in logfile.pdf look sensible?  > In order to overcome this, I tried \\nseveral approache s: I tried  polishing with and without training on the particles, I regrouped them,  I \\nchanged box size, I changed workstation and I also did the job on the  whole particle and on a focused \\narea (the particle is a ribosome). I  also tried to make the refin ement process easier for Relion, keeping \\nthe  reference not so much low -pass filtered (at this stage it is difficult  to bias the result), adding \\nmaxsig 3000 as additional argument, using a  mask and skipping padding and gridding.  > Nothing of \\nthis worked  and I don t know how to proceed. Polishing preserves particle alignment parameters in the \\ninput STAR file. Run relion_reconstruct on shiny.star to make half set reconstructions and perform \\nPostProcess. If the resolution is similar to that before Refine3D,  your polished particles are fine and \\nRefine3D is struggling. Otherwise, the output of Polish is already bad. It is also worth running Class2D on \\nshiny particles. Sometimes detector defects come back after Polish. Such particles should be removed. \\nBest reg ards, Takanori Nakane On 6/29/23 22:07, Martino Morici wrote: > Hi everyone >  > I am \\nprocessing a dataset in Relion 4 that is giving me some issues. > After CTF refining, the particles were not \\naligning anymore upon 3D refinement. Therefore I tried 3D -refining after each CTF step and I figured out \\nthe problem was the 4th order aberrations paramenter: when set to No, refinement was then working \\ncorrectly (I didn t use trefoil estimation). > However, after I do bayesian polishing, the 3D refinement \\ngives the  same problem, i.e. it is not able to align the particles (the job results in a random bowl of \\ndensity, empty inside. > In order to overcome this, I tried several approaches: I tried polishing with and \\nwithout training on the particles, I regrouped them, I  changed box size, I changed workstation and I also \\ndid the job on the whole particle and on a focused area (the particle is a ribosome). I also tried to make \\nthe refinement process easier for Relion, keeping the reference not so much low -pass filtered (at  this \\nstage it is difficult to bias the result), adding maxsig 3000 as additional argument, using a mask and \\nskipping padding and gridding. > Nothing of this worked and I don t know how to proceed. >  > Has \\nanyone ever had this kind of problem and know how  to solve it? > Thank you for your time > - \\nMartino >  > \\n######################################################################## >  > To \\nunsubscribe from the CCPEM list, click the following link: > \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCP EM&A=1 >  > This message was issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing list hosted by www.jiscmail.ac.uk, terms & conditions \\nare available at https://www.jiscmail.ac.uk/policyandsecurity/ \\n##################################################### ################### To unsubscribe \\nfrom the CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message was issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing list hosted by www.jiscmail.ac.uk, t erms & conditions \\nare available at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print Hi, Please \\ndon't drop CC to CCPEM.  > If possible, could you kindly elaborate  > a little on the problem with \\nsaturating A40 cards with RELION.  does this  > have to do with the box size, for ex? GPU is a massively \\nparallel processor. Ideally, the processing performance should increase linearly with the number of (GPU) \\ncores. To achieve this, all cores have to perform processing tasks simultaneous ly and nobody should \", \"remain idle. Unfortunately, RELION's implementation do not have such a high level of parallelism to \\nutilize all cores in the latest GPUs. For example, multiple tasks cannot accumulate into a single voxel \\n(called a race condition) durin g back -projection. While one task writes, all others have to wait. One way \\nto mitigate this is to run multiple RELION processes on a GPU. But this multiplies GPU memory \\nrequirements. This is OK for A40, which has 48 GB of RAM, but the memory bandwidth then  becomes \\nlimiting. Also note that more MPI processes need more CPU powers and coordination overheads among \\nMPI processes start to grow. In my opinion, A5000 is well balanced among Tesla and Quadro cards. I \\nwould run one or two MPI processes on a A5000. Of course the sweet spot depends on the box size, \\nresolution etc. If your group always work on huge virus datasets, it is a different story. But for general, \\ndiverse workloads expected in a shared facility, A5000 should be adequate. That being said, I haven't  \\ntested all cards. You should run benchmarks of realistic workloads (not our old Ribosome Class3D \\nbenchmark). Most HPC vendors let you test -drive their system before placing orders. Best regards, \\nTakanori Nakane On 6/29/23 23:00, Eaazhisai KANDIAH wrote: >  Dear Takanori, >  > Thanks very much \\nfor your input. If possible, could you kindly elaborate  > a little on the problem with saturating A40 \\ncards with RELION. does this  > have to do with the box size, for ex? A5000 cards are in the menu as  > \\nwell. Some colleagues also recommended A6000 cards, but i guess the  > difference might be little \\nbetween these two. >  > A load of thanks again, >  > Best >  > Isai >  > <mailto:[log in to \\nunmask]> >  > On 29/06/2023 04:00, Takanori Nakane wrote: >> Hi, >> >> > Do R ELION and CRYOSPARC \\ntake advantage on the capabilities offered  >> by NVLink. >> >> RELION don't use NVLink. >> >> > Lastly, \\nare there any alternative >> > GPU cards currently available that offer similar functionality to the >> > \\nA40 but at a more afforda ble price? >> >> Assuming you want to avoid GeForce cards, my \\nrecommendation for >> RELION (I cannot comment on other programs) is NVIDIA A5000. >> >> A40 is \\nvery hard to saturate with RELION, both in terms >> of single precision FLOPS and GPU memory. >> Y ou \\ncan buy two A5000s for the price of one A40 and this should give >> better aggregate throughput. >> >> \\nFYI, we are building a GPU cluster for SPA. >> One node will have 4 x A5000s, 512 GB of RAM and >> 64 \\nCPU cores (128 threads). >> One node will host t wo jobs (2 GPUs each, divided by cgroups) >> or one job \\n(4 GPUs; worse total throughput but lower run -time). >> >> Best regards, >> >> Takanori Nakane >> >> \\nOn 2023/06/29 5:29, Eaazhisai KANDIAH wrote: >>> Dear all, >>> >>> I would appreciate suggestions o r \\nreferences to benchmark studies  >>> regarding the utilization of A40 GPU cards in a workstation \\nto  >>> execute RELION and CRYOSPARC. Given that A40 cards are typically  >>> recommended for \\nservers due to their passive cooling, I am concerned  >>> about  potential heating problems if they are \\nemployed in a  >>> workstation. Any extra fittings are needed in the chassis to adopt  >>> these in a \\nworkstation? Additionally, for a workstation equipped with  >>> 4 or 8 GPUs, would it be advantageous \\nto opt for N VLink? Do RELION  >>> and CRYOSPARC take advantage on the capabilities offered by \\nNVLink.  >>> Lastly, are there any alternative GPU cards currently available that  >>> offer similar \\nfunctionality to the A40 but at a more affordable price? >>> >>> Thanks s o much in advance for your \\ntime and help, >>> >>> Best >>> >>> Isai >>> >>> --  >>> >>> *Dr. Eaazhisai Kandiah* >>> >>> CM01 \\nbeamline >>> >>> https://www.esrf.fr/CM01 >>> >>> 71 Avenue des Martyrs, 38000 Grenoble >>> \\nTelephone: +33 476 882691 >>> Email: [l og in to unmask] <mailto:[log in to unmask]> >>> >>> >>> \\n------------------------------------------------------------------------  >>> >>> To unsubscribe from the CCPEM list, click \\nthe following link: >>> https://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED 1=CCPEM&A=1  >>> \\n<https://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1> >>> \\n######################################################################## To unsubscribe \\nfrom the CCPEM list, click the following link: \", \"https://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message was issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing list hosted by www.jiscmail.ac.uk, terms & conditions \\nare available at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print Hi \\neveryone I am processing a dataset in Relion 4 that is giving me some issues. After CTF refining, the \\nparticles were not aligning anymore upon 3D refinement. Therefore I tried 3D -refining after each CTF \\nstep and I figured out the problem was the 4th order aberrations paramenter: when set to No, \\nrefinement was then working correctly (I didn t use trefoil estimation). However, after I do bayesian \\npolishing, the 3D refinement gives the same problem, i.e. it is not able to align the particles (the job \\nresults i n a random bowl of density, empty inside. In order to overcome this, I tried several approaches: I \\ntried polishing with and without training on the particles, I regrouped them, I changed box size, I \\nchanged workstation and I also did the job on the whole p article and on a focused area (the particle is a \\nribosome). I also tried to make the refinement process easier for Relion, keeping the reference not so \\nmuch low -pass filtered (at this stage it is difficult to bias the result), adding maxsig 3000 as additio nal \\nargument, using a mask and skipping padding and gridding. Nothing of this worked and I don t know \\nhow to proceed. Has anyone ever had this kind of problem and know how to solve it? Thank you for your \\ntime - Martino ##################################### ################################### \\nTo unsubscribe from the CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message was issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing list hosted by www.j iscmail.ac.uk, terms & conditions \\nare available at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print Hi,  > Do \\nRELION and CRYOSPARC take advantage on the capabilities offered by  NVLink. RELION don't use \\nNVLink.  > Lastly, are t here any alternative  > GPU cards currently available that offer similar \\nfunctionality to the  > A40 but at a more affordable price? Assuming you want to avoid GeForce cards, \\nmy recommendation for RELION (I cannot comment on other programs) is NVIDIA A5000 . A40 is very \\nhard to saturate with RELION, both in terms of single precision FLOPS and GPU memory. You can buy two \\nA5000s for the price of one A40 and this should give better aggregate throughput. FYI, we are building a \\nGPU cluster for SPA. One node will have 4 x A5000s, 512 GB of RAM and 64 CPU cores (128 threads). One \\nnode will host two jobs (2 GPUs each, divided by cgroups) or one job (4 GPUs; worse total throughput \\nbut lower run -time). Best regards, Takanori Nakane On 2023/06/29 5:29, Eaazhisai KANDIAH  wrote: > \\nDear all, >  > I would appreciate suggestions or references to benchmark studies  > regarding the \\nutilization of A40 GPU cards in a workstation to execute  > RELION and CRYOSPARC. Given that A40 \\ncards are typically recommended for  > servers due to their passive cooling, I am concerned about \\npotential  > heating problems if they are employed in a workstation. Any extra  > fittings are needed in \\nthe chassis to adopt these in a workstation?  > Additionally, for a workstation equipped with 4 or 8 \\nGPU s, would it be  > advantageous to opt for NVLink? Do RELION and CRYOSPARC take advantage  > \\non the capabilities offered by NVLink. Lastly, are there any alternative  > GPU cards currently available \\nthat offer similar functionality to the  > A40 but at a mo re affordable price? >  > Thanks so much in \\nadvance for your time and help, >  > Best >  > Isai >  > --  >  > *Dr. Eaazhisai Kandiah* >  > CM01 \\nbeamline >  > https://www.esrf.fr/CM01 >  > 71 Avenue des Martyrs, 38000 Grenoble > Telephone: \\n+33 476 882691 > Email: [log in to unmask] <mailto:[log in to unmask]> >  >  > \\n------------------------------------------------------------------------  >  > To unsubscribe from the CCPEM list, click \\nthe following link: > https://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED 1=CCPEM&A=1  > \\n<https://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1> >  \\n######################################################################## To unsubscribe \", \"from the CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message was issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing list hosted by www.jiscmail.ac.uk, terms & conditions \\nare available at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print Dear a ll, I \\nwould appreciate suggestions or references to benchmark studies  regarding the utilization of A40 GPU \\ncards in a workstation to execute  RELION and CRYOSPARC. Given that A40 cards are typically \\nrecommended for  servers due to their passive cooling, I  am concerned about potential  heating \\nproblems if they are employed in a workstation. Any extra  fittings are needed in the chassis to adopt \\nthese in a workstation?  Additionally, for a workstation equipped with 4 or 8 GPUs, would it be  \\nadvantageous to o pt for NVLink? Do RELION and CRYOSPARC take advantage  on the capabilities \\noffered by NVLink. Lastly, are there any alternative  GPU cards currently available that offer similar \\nfunctionality to the  A40 but at a more affordable price? Thanks so much in ad vance for your time and \\nhelp, Best Isai --  *Dr. Eaazhisai Kandiah* CM01 beamline https://www.esrf.fr/CM01 71 Avenue des \\nMartyrs, 38000 Grenoble Telephone: +33 476 882691 Email: [log in to unmask] <mailto:[log in to \\nunmask]> ############################### ######################################### To \\nunsubscribe from the CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message was issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing list hosted by  www.jiscmail.ac.uk, terms & conditions \\nare available at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print Hello, \\nNanoSoft is developer, manufacturer and provider of tools, instruments and consumables for cryoEM \\nsample preparati on, handling and storage. We are now producing and selling autogrid rings and c -clips \\nfor use in autoloader cryoEM systems. We are using the most cutting edge manufacturing technology to \\ncreate premium parts with low variability and selling them at half th e price \\n(https://www.nanosoftmaterials.com/clippingandautogridrings). They've now been tested by 50+ labs \\nwithout a single issue/defect. They've worked in both autoloaders and in the column without issue, and \\nimages processed with our autogrids have result ed in high quality data. Please reach out with any \\nquestions. Thanks, Mike ([log in to unmask]) Michael Godfrin, PhD President 1372 Main St. Coventry, RI \\n02816 Cell: (401) 249 -0780 www.nanosoftmaterials.com \\n################################################# ####################### To unsubscribe \\nfrom the CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message was issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing list hosted by www.jiscmail.ac.u k, terms & conditions \\nare available at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print Our \\nresearch group at Karolinska Institutet (http://jovinelab.org) is looking for two ambitious postdoctoral \\nfellows who would like to spea rhead our future steps towards unveiling the molecular basis of gamete \\nrecognition and fusion through the evolutionary tree. For more information and to submit an application \\n(deadline 31 July 2023), please visit: Integrative Structural Biology of Fertiliz ation \\nhttps://ki.varbi.com/se/what:job/jobID:637984 Biochemistry of Fertilization \\nhttps://ki.varbi.com/se/what:job/jobID:639567 Of course, you are also always welcome to contact me \\ndirectly for informal queries. We look forward to receiving your applicatio n! With best regards, Luca -- \\nLuca Jovine, Ph.D. Professor of Structural Biology, Member of EMBO and the Nobel Assembly Karolinska \\nInstitutet Department of Biosciences and Nutrition Medicinaren 25 Neo Blickag ngen 16, SE -141 83 \\nHuddinge, Sweden E -mail: [lo g in to unmask] W3: http://jovinelab.org N r du skickar e -post till Karolinska \\nInstitutet (KI) inneb r detta att KI kommer att behandla dina personuppgifter. H r finns information om \", 'hur KI behandlar personuppgifter<https://ki.se/medarbetare/integritetssky ddspolicy>. Sending email to \\nKarolinska Institutet (KI) will result in KI processing your personal data. You can read more about KI s \\nprocessing of personal data here<https://ki.se/en/staff/data -protection -policy>. \\n######################################### ############################### To unsubscribe \\nfrom the CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message was issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing list hosted by www.jiscm ail.ac.uk, terms & conditions \\nare available at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print Hi, The \\nTinghai Xu Laboratory is currently seeking highly motivated and independent postdoctoral fellows with a \\nrecent Ph.D. in the  field of biochemistry and molecular biology. The primary focus of these positions is to \\nuncover the mysteries of mammalian DNA methylation in the context of chromatin using advanced \\nstructural biology and cell biology techniques. We employ a range of stru ctural and biochemical methods, \\nincluding single -particle cryo -EM, protein -protein interaction in vitro, and in -cell, in -solution structure \\ndetermination. The cryo -EM facility at WUCCI (https://wucci.wustl.edu/) hosts a Titan Krios G3, a Glacios, \\nand an Aq uilos 2 Cryo -FIB. For more information, please visit our lab website: \\nhttps://www.xulaboratory.org/ Qualifications:   *   Ph.D. in biochemistry, molecular biology, or \\nbiology -related fields.   *   Experience in protein engineering, expression, purification , and \\nbiochemical characterization.   *   Structural biology experience and DNA methylation experience \\nwould be a plus. We encourage candidates who fulfill these qualifications and demonstrate a passion for \\nscience to apply for these positions. Our postdoc toral positions offer an exciting and dynamic \\nenvironment, providing ample opportunities for professional development and expertise in advanced \\ntechniques of structural and cell biology techniques. Please send the following documents to Dr. Tinghai \\nXu: [lo g in to unmask]<mailto:[log in to unmask]>.   *   Cover letter.   *   Current curriculum vitae.   \\n*   Contact information of three references.   *   Concise summary of past research \\naccomplishments and future research interests. Who is Saint Louis Universi ty? Founded in 1818, Saint \\nLouis University is one of the nation s oldest and most prestigious Catholic universities. SLU, which also \\nhas a campus in Madrid, Spain, is recognized for world -class academics, life -changing research, \\ncompassionate health care,  and a strong commitment to faith and service. \\n######################################################################## To unsubscribe \\nfrom the CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This messa ge was issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing list hosted by www.jiscmail.ac.uk, terms & conditions \\nare available at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print Thank \\nyou Takanori. The subtraction is now running without increasing memory usage. Everybody, the solution \\nis to change the value of this environmental variable: export \\nGLIBC_TUNABLES=glibc.malloc.mmap_threshold=1048576 Thanks a lot and best regards, Yaroslav \\n-----Original Message ----- From: Colla borative Computational Project in Electron cryo -Microscopy <[log in \\nto unmask]> On Behalf Of Takanori Nakane Sent: Tuesday, June 27, 2023 9:14 PM To: [log in to unmask] \\nSubject: Re: [ccpem] [EXTERNAL] Re: [ccpem] Relion 4 runs out of memory during particle  subtraction Hi, \\nA certain version of memory allocator in GLIBC is not happy with RELION. This is not a memory leak (i.e. \\nbug), but memory fragmentation. See \\nhttps://github.com/3dem/relion/issues/673#issuecomment -773177455 for the details and a \\nworkaround.  Best regards, Takanori Nakane On 2023/06/28 9:59, Reza Khayat wrote: >  Hi, > > > Solve \\nthe problem a different way. Break your star file into multiple > star files. Process each star file separately. ', 'Then combine all the > star files for your next step. > > > Best wishes, > > Reza > > > Reza Khayat, PhD > \\nAssociate Professor > City College of New York > Department of Chemistry and Biochemistry New York, \\nNY 10031 > ----------------------------------------------------------------------  > -- > *From:* Collab orative \\nComputational Project in Electron > cryo -Microscopy <[log in to unmask]> on behalf of Tsybovsky, > \\nYaroslav (NIH/NCI) [C] <[log in to unmask]> > *Sent:* Tuesday, June 27, 2023 8:56 PM > *To:* [log in to \\nunmask] > *Subject:* Re: [ccpem] [EXTERNAL] R e: [ccpem] Relion 4 runs out of > memory during particle \\nsubtraction > > Hi Dieter, > > Thanks for the hint. Unfortunately, in my case crashes seem to happen > \\nirrespectively of the number of mpi processes. I tried 2, 4, 6, 8, 12, > 24 and 32. When the num ber is up \\nto 16, the job crashes during the >  Subtracting all particles  stage after having finished about 5% of > \\nthe work. When the number of mpi processes is over 16, the job crashes > before starting the actual \\nsubtraction (at the  Calculating Fourier  > transforms of the map  stage). > > Thanks again, > > \\nYaroslav > > *From:* Dieter Blaas <[log in to unmask]> > *Sent:* Tuesday, June 27, 2023 4:02 PM > *To:* \\nTsybovsky, Yaroslav (NIH/NCI) [C] <[log in to unmask]>; > [log in to unmask] > *Subject:* [EXTER NAL] Re: \\n[ccpem] Relion 4 runs out of memory during > particle subtraction > > Hi Yaroslav, > >      you might \\nlook up my previous correspondence. Takanori drew my > attention to the fact that I had to use more \\nthreads to not run out of > RAM. For example this finally worked on our cluster: > > #SBATCH \\n--job-name=Relion4Job > *#SBATCH --ntasks=24 > #SBATCH --cpus -per-task=1 > #SBATCH \\n--mem -per-cpu=12G* > #SBATCH --partition=basic > #SBATCH --output=Subtract/job027/run.out \\n#SBATCH > --error=Subtract/job027/r un.err #SBATCH --time=2 -00:00:00 > > I am using relion -4 > > best, \\nDieter > > ----------------------------------------------------------------------  > -- > > Dieter Blaas, > > Max Perutz \\nLaboratories > > Medical University of Vienna, > > Inst. Med. Biochem ., Vienna Biocenter (VBC), > > Dr. \\nBohr Gasse 9/3, > > A -1030 Vienna, Austria, > > Tel: 0043 1 4277 61630, > > Mobile: 0043 699 1942 \\n1659 > > e -mail:[log in to unmask] > <mailto:[log in to unmask]> > > \\n------------------------------------------------------ ----------------  > -- > > On 27.06.2023 20:58, Tsybovsky, Yaroslav \\n(NIH/NCI) [C] wrote: > >     Hello everybody, > >     I am trying to do particle subtraction in Relion \\n4.0.1 using two mpi >     processes on a workstation with 512 Gb of RAM. I am seeing t he two >     \\nmpi processes consuming progressively more RAM (about 25% each in >     the attached screenshot, \\nor 200Gb in total) until they run out of >     RAM about half an hour into the subtraction, upon which \\nthey are >     killed by the OS. Does anybo dy know why this happens? I have never >     \\nencountered this with Relion 3. > >     Thank you, > >     Yaroslav > >     Yaroslav Tsybovsky, Ph.D. \\n(contractor) > >     Senior Scientist & Head > >     Vaccine Research Center Electron Microscopy \\nUnit > >     Frederick National Laboratory for Cancer Research > >     Leidos Biomedical Research, \\nInc. > >     Post Office Box B > >     Frederick, Maryland 21702 > >     Phone: 301 -846-7550 > > > \\n----------------------------------------------------------------------  > -- > >     To unsubscribe from the CCPEM list, \\nclick the following link: >     Bad URL Removed - see why - > > \\nhttps://ees.sps.nih.gov/services/Pages/Anti -Virus.aspx?SUBED1=CCPEM&A= > 1 > > CAUTION:This email \\noriginated from outside of the organization.  Do not > click links or open attachments unless you \\nrecognize the sender and > are confident the content is safe. > > > \\n----------------------------------------------------------------------  > -- > > To unsubscribe from the CCPEM list, click \\nthe following  link: > Bad URL Removed - see why - > \\nhttps://ees.sps.nih.gov/services/Pages/Anti -Virus.aspx?SUBED1=CCPEM&A= > 1 <Bad URL Removed - \\nsee why - > https://ees.sps.nih.gov/services/Pages/Anti -Virus.aspx -3FSUBED1 -3DCCPE > \\nM-26A-3D1&d=DwMGaQ&c=4NmamNZG3KTnUCoC6 InoLJ6KV1tbVKrkZXHRwtIMGmo&r=1Dz > \\nJFW0v6TgEhkW1gy_ -ke-RbtvS1fzEbD5_hcb9Up0&m=K3VcQVTTFgR2XMvbM6dURkGMALL > ', 'ZqdYC0T14uJVIja -yLC9UnSrTfEYEZYg13_X1&s=5SBvMepVZDNRtGtWrkdlPyerAUhLko > \\n8G-ulJPeWociU&e=> > > > --------------------------------------------------- -------------------  > -- > > To unsubscribe \\nfrom the CCPEM list, click the following link: > Bad URL Removed - see why - > \\nhttps://ees.sps.nih.gov/services/Pages/Anti -Virus.aspx?SUBED1=CCPEM&A= > 1 <Bad URL Removed - \\nsee why - > https://ees.sps.nih.gov/ser vices/Pages/Anti -Virus.aspx?SUBED1=CCPEM&A= > 1> > \\n######################################################################## To unsubscribe \\nfrom the CCPEM list, click the following link: Bad URL Removed - see why - \\nhttps://ees.sps.nih.gov/services/Pages/Ant i-Virus.aspx?SUBED1=CCPEM&A=1 This message was issued \\nto members of http://www.jiscmail.ac.uk/CCPEM, a mailing list hosted by http://www.jiscmail.ac.uk/, \\nterms & conditions are available at https://www.jiscmail.ac.uk/policyandsecurity/ CAUTION: This email \\noriginated from outside of the organization. Do not click links or open attachments unless you recognize \\nthe sender and are confident the content is safe. \\n######################################################################## To unsubscribe \\nfrom the CCPE M list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message was issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing list hosted by www.jiscmail.ac.uk, terms & conditions \\nare available at https://www.j iscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print Hi, A \\ncertain version of memory allocator in GLIBC is not happy with RELION. This is not a memory leak (i.e. \\nbug), but memory fragmentation. See \\nhttps://github.com/3dem/relion/issues/673#issu ecomment -773177455 for the details and a \\nworkaround. Best regards, Takanori Nakane On 2023/06/28 9:59, Reza Khayat wrote: >  Hi, >  >  > \\nSolve the problem a different way. Break your star file into multiple  > star files. Process each star file \\nseparately.  Then combine all the star  > files for your next step. >  >  > Best wishes, >  > \\nReza >  >  > Reza Khayat, PhD > Associate Professor > City College of New York > Department of \\nChemistry and Biochemistry > New York, NY 10031 > \\n----------------------------- -------------------------------------------  > *From:* Collaborative Computational Project in \\nElectron cryo -Microscopy  > <[log in to unmask]> on behalf of Tsybovsky, Yaroslav (NIH/NCI) [C]  > \\n<[log in to unmask]> > *Sent:* Tuesday, June 27, 2023 8:56 PM > *To:* [log in to unmask] > *Subject:* \\nRe: [ccpem] [EXTERNAL] Re: [ccpem] Relion 4 runs out of  > memory during particle subtraction >  > Hi \\nDieter, >  > Thanks for the hint. Unfortunately, in my case crashes seem to happen  > irrespectively of \\nthe number o f mpi processes. I tried 2, 4, 6, 8, 12,  > 24 and 32. When the number is up to 16, the job \\ncrashes during the  >  Subtracting all particles  stage after having finished about 5% of the  > work. \\nWhen the number of mpi processes is over 16, the job crashes  > before starting the actual subtraction \\n(at the  Calculating Fourier  > transforms of the map  stage). >  > Thanks again, >  > Yaroslav >  > \\n*From:* Dieter Blaas <[log in to unmask]> > *Sent:* Tuesday, June 27, 2023 4:02 PM > *To:* Tsybovsky, \\nYaroslav (N IH/NCI) [C] <[log in to unmask]>;  > [log in to unmask] > *Subject:* [EXTERNAL] Re: [ccpem] \\nRelion 4 runs out of memory during  > particle subtraction >  > Hi Yaroslav, >  >    you might look \\nup my previous correspondence. Takanori drew my  > attention to the fact that I had to use more \\nthreads to not run out of  > RAM. For example this finally worked on our cluster: >  > #SBATCH \\n--job-name=Relion4Job > *#SBATCH --ntasks=24 > #SBATCH --cpus -per-task=1 > #SBATCH \\n--mem -per-cpu=12G* > #SBATCH --partition=basic  > #SBATCH --output=Subtract/job027/run.out > \\n#SBATCH --error=Subtract/job027/run.err > #SBATCH --time=2 -00:00:00 >  > I am using relion -4 >  > \\nbest, Dieter >  > ------------------------------------------------------------------------  >  > Dieter Blaas, >  > Max \\nPerutz Laboratories >  > Medical University of Vienna, >  > Inst. Med. Biochem., Vienna Biocenter ', '(VBC), >  > Dr. Bohr Gasse 9/3, >  > A -1030 Vienna, Austria, >  > Tel: 0043 1 4277 61630, >  > Mobile: \\n0043 699 1942 1659 >  > e -mail:[log in to unmask ]  <mailto:[log in to unmask]> >  > \\n------------------------------------------------------------------------  >  > On 27.06.2023 20:58, Tsybovsky, Yaroslav \\n(NIH/NCI) [C] wrote: >  >     Hello everybody, >  >     I am trying to do particle subtraction in \\nRelion 4.0.1 using two mpi >     processes on a workstation with 512 Gb of RAM. I am seeing the \\ntwo >     mpi processes consuming progressively more RAM (about 25% each in >     the attached \\nscreenshot, or 200Gb in total) until they run out of >     RAM about  half an hour into the subtraction, \\nupon which they are >     killed by the OS. Does anybody know why this happens? I have never >     \\nencountered this with Relion 3. >  >     Thank you, >  >     Yaroslav >  >     Yaroslav Tsybovsky, \\nPh.D. (contractor) >  >     Senior Scientist & Head >  >     Vaccine Research Center Electron \\nMicroscopy Unit >  >     Frederick National Laboratory for Cancer Research >  >     Leidos \\nBiomedical Research, Inc. >  >     Post Office Box B >  >     Frederick, Maryland 21702 >  >     \\nPhone: 301 -846-7550 >  >     ------------------------------------------------------------------------  >  >     To \\nunsubscribe from the CCPEM list, click the following link: >     Bad URL Removed - see why - >     \\nhttps://ees.sps.nih.gov/services/Pages/ Anti-Virus.aspx?SUBED1=CCPEM&A=1 >  > CAUTION:This email \\noriginated from outside of the organization. Do not  > click links or open attachments unless you \\nrecognize the sender and are  > confident the content is safe. >  >  > \\n------------------------------ ------------------------------------------  >  > To unsubscribe from the CCPEM list, click \\nthe following link: > https://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1  > \\n<https://urldefense.proofpoint.com/v2/url?u=https -3A__www.jiscmail.ac.uk_cgi -2Dbin_WA -2DJISC.exe -\\n3FSUBED1 -3DCCPEM -26A-3D1&d=DwMGaQ&c=4NmamNZG3KTnUCoC6InoLJ6KV1tbVKrkZXHRwtIMGmo\\n&r=1DzJFW0v6TgEhkW1gy_ -ke-RbtvS1fzEbD5_hcb9Up0&m=K3VcQVTTFgR2XMvbM6dURkGMALLZqdYC\\n0T14uJVIja -yLC9UnSrTfEYEZYg13_X1&s=5SBvMepVZDNRtGtWrkdlPyerAUhLko8G -ulJPeWoc iU&e=> >  >\\n  > ------------------------------------------------------------------------  >  > To unsubscribe from the CCPEM list, \\nclick the following link: > https://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1  > \\n<https://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1> >  \\n######################################################################## To unsubscribe \\nfrom the CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message was iss ued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing list hosted by www.jiscmail.ac.uk, terms & conditions \\nare available at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print ?Hi, Solve \\nthe problem a different way. Break your st ar file into multiple star files. Process each star file separately. \\nThen combine all the star files for your next step. Best wishes, Reza Reza Khayat, PhD Associate Professor \\nCity College of New York Department of Chemistry and Biochemistry New York, NY 1 0031 \\n________________________________ From: Collaborative Computational Project in Electron \\ncryo -Microscopy <[log in to unmask]> on behalf of Tsybovsky, Yaroslav (NIH/NCI) [C] <[log in to unmask]> \\nSent: Tuesday, June 27, 2023 8:56 PM To: [log in to unmask]  Subject: Re: [ccpem] [EXTERNAL] Re: [ccpem] \\nRelion 4 runs out of memory during particle subtraction Hi Dieter, Thanks for the hint. Unfortunately, in \\nmy case crashes seem to happen irrespectively of the number of mpi processes. I tried 2, 4, 6, 8, 12, 24 \\nand 32. When the number is up to 16, the job crashes during the \"Subtracting all particles...\" stage after \\nhaving finished about 5% of the work. When the number of mpi processes is over 16, the job crashes \\nbefore starting the actual subtraction (at the \"Ca lculating Fourier transforms of the map...\" stage). \\nThanks again, Yaroslav From: Dieter Blaas <[log in to unmask]> Sent: Tuesday, June 27, 2023 4:02 PM To: \\nTsybovsky, Yaroslav (NIH/NCI) [C] <[log in to unmask]>; [log in to unmask] Subject: [EXTERNAL] Re: ', '[ccpem] Relion 4 runs out of memory during particle subtraction Hi Yaroslav,     you might look up my \\nprevious correspondence. Takanori drew my attention to the fact that I had to use more threads to not \\nrun out of RAM. For example this finally worked on ou r cluster: #SBATCH --job-name=Relion4Job \\n#SBATCH --ntasks=24 #SBATCH --cpus -per-task=1 #SBATCH --mem -per-cpu=12G #SBATCH \\n--partition=basic #SBATCH --output=Subtract/job027/run.out #SBATCH --error=Subtract/job027/run.err \\n#SBATCH --time=2 -00:00:00 I am using  relion -4 best, Dieter \\n------------------------------------------------------------------------  Dieter Blaas, Max Perutz Laboratories Medical \\nUniversity of Vienna, Inst. Med. Biochem., Vienna Biocenter (VBC), Dr. Bohr Gasse 9/3, A -1030 Vienna, \\nAustria, Tel : 0043 1 4277 61630, Mobile: 0043 699 1942 1659 e -mail: [log in to unmask]<mailto:[log in to \\nunmask]> ------------------------------------------------------------------------  On 27.06.2023 20:58, Tsybovsky, \\nYaroslav (NIH/NCI) [C] wrote: Hello everybody, I am trying to do particle subtraction in Relion 4.0.1 using \\ntwo mpi processes on a workstation with 512 Gb of RAM. I am seeing the two mpi processes consuming \\nprogressively more RAM (about 25% each in the attached screenshot, or 200Gb in total) until they r un \\nout of RAM about half an hour into the subtraction, upon which they are killed by the OS. Does anybody \\nknow why this happens? I have never encountered this with Relion 3. Thank you, Yaroslav Yaroslav \\nTsybovsky, Ph.D. (contractor) Senior Scientist & Head  Vaccine Research Center Electron Microscopy Unit \\nFrederick National Laboratory for Cancer Research Leidos Biomedical Research, Inc. Post Office Box B \\nFrederick, Maryland 21702 Phone: 301 -846-7550 ________________________________ To unsubscribe \\nfrom the CC PEM list, click the following link: Bad URL Removed - see why - \\nhttps://ees.sps.nih.gov/services/Pages/Anti -Virus.aspx?SUBED1=CCPEM&A=1 CAUTION: This email \\noriginated from outside of the organization. Do not click links or open attachments unless you recog nize \\nthe sender and are confident the content is safe. ________________________________ To unsubscribe \\nfrom the CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1<https://urldefense.proofpoint.co\\nm/v2/url?u =https -3A__www.jiscmail.ac.uk_cgi -2Dbin_WA -2DJISC.exe -3FSUBED1 -3DCCPEM -26A-3D1&d\\n=DwMGaQ&c=4NmamNZG3KTnUCoC6InoLJ6KV1tbVKrkZXHRwtIMGmo&r=1DzJFW0v6TgEhkW1gy_ -ke-R\\nbtvS1fzEbD5_hcb9Up0&m=K3VcQVTTFgR2XMvbM6dURkGMALLZqdYC0T14uJVIja -yLC9UnSrTfEYEZYg13_\\nX1&s=5SBvMep VZDNRtGtWrkdlPyerAUhLko8G -ulJPeWociU&e=> \\n######################################################################## To unsubscribe \\nfrom the CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message was  issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing list hosted by www.jiscmail.ac.uk, terms & conditions \\nare available at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print Hi Dieter, \\nThanks for the hint. Unfortunately, in  my case crashes seem to happen irrespectively of the number of \\nmpi processes. I tried 2, 4, 6, 8, 12, 24 and 32. When the number is up to 16, the job crashes during the  \\nSubtracting all particles  stage after having finished about 5% of the work. When the  number of mpi \\nprocesses is over 16, the job crashes before starting the actual subtraction (at the  Calculating Fourier \\ntransforms of the map  stage). Thanks again, Yaroslav From: Dieter Blaas <[log in to unmask]> Sent: \\nTuesday, June 27, 2023 4:02 PM To: Tsybovsky, Yaroslav (NIH/NCI) [C] <[log in to unmask]>; [log in to \\nunmask] Subject: [EXTERNAL] Re: [ccpem] Relion 4 runs out of memory during particle subtraction Hi \\nYaroslav,     you might look up my previous correspondence. Takanori drew my attention to the fact \\nthat I had to use more threads to not run out of RAM. For example this finally worked on our cluster: \\n#SBATCH --job-name=Relion4Job #SBATCH --ntasks=24 #SBATCH --cpus -per-task=1 #SBATCH \\n--mem -per-cpu=12G #SBATCH --partition=basic #SBATCH --output= Subtract/job027/run.out #SBATCH ', '--error=Subtract/job027/run.err #SBATCH --time=2 -00:00:00 I am using relion -4 best, Dieter \\n------------------------------------------------------------------------  Dieter Blaas, Max Perutz Laboratories Medical \\nUniversity of Vienna, Inst. Med. Biochem., Vienna Biocenter (VBC), Dr. Bohr Gasse 9/3, A -1030 Vienna, \\nAustria, Tel: 0043 1 4277 61630, Mobile: 0043 699 1942 1659 e -mail: [log in to unmask]<mailto:[log in to \\nunmask]> ------------------------------------------------------ ------------------  On 27.06.2023 20:58, Tsybovsky, \\nYaroslav (NIH/NCI) [C] wrote: Hello everybody, I am trying to do particle subtraction in Relion 4.0.1 using \\ntwo mpi processes on a workstation with 512 Gb of RAM. I am seeing the two mpi processes consumin g \\nprogressively more RAM (about 25% each in the attached screenshot, or 200Gb in total) until they run \\nout of RAM about half an hour into the subtraction, upon which they are killed by the OS. Does anybody \\nknow why this happens? I have never encountered th is with Relion 3. Thank you, Yaroslav Yaroslav \\nTsybovsky, Ph.D. (contractor) Senior Scientist & Head Vaccine Research Center Electron Microscopy Unit \\nFrederick National Laboratory for Cancer Research Leidos Biomedical Research, Inc. Post Office Box B \\nFrede rick, Maryland 21702 Phone: 301 -846-7550 ________________________________ To unsubscribe \\nfrom the CCPEM list, click the following link: Bad URL Removed - see why - \\nhttps://ees.sps.nih.gov/services/Pages/Anti -Virus.aspx?SUBED1=CCPEM&A=1<Bad%20URL%20Removed\\n%20-%20see%20why%20 -%20https:/ees.sps.nih.gov/services/Pages/Anti -Virus.aspx?SUBED1=CCPEM&A\\n=1> CAUTION: This email originated from outside of the organization. Do not click links or open \\nattachments unless you recognize the sender and are confident the cont ent is safe. \\n######################################################################## To unsubscribe \\nfrom the CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message was issued to \\nmembers of www.ji scmail.ac.uk/CCPEM, a mailing list hosted by www.jiscmail.ac.uk, terms & conditions \\nare available at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print Hi \\nYaroslav,    you might look up my previous correspondence. Takanori drew m y  attention to the fact \\nthat I had to use more threads to not run out of  RAM. For example this finally worked on our cluster: \\n#SBATCH --job-name=Relion4Job *#SBATCH --ntasks=24** **#SBATCH --cpus -per-task=1** **#SBATCH \\n--mem -per-cpu=12G* #SBATCH --partit ion=basic #SBATCH --output=Subtract/job027/run.out #SBATCH \\n--error=Subtract/job027/run.err #SBATCH --time=2 -00:00:00 I am using relion -4 best, Dieter \\n------------------------------------------------------------------------  Dieter Blaas, Max Perutz Laborato ries Medical \\nUniversity of Vienna, Inst. Med. Biochem., Vienna Biocenter (VBC), Dr. Bohr Gasse 9/3, A -1030 Vienna, \\nAustria, Tel: 0043 1 4277 61630, Mobile: 0043 699 1942 1659 e -mail:[log in to unmask] \\n------------------------------------------------------- -----------------  On 27.06.2023 20:58, Tsybovsky, Yaroslav \\n(NIH/NCI) [C] wrote: > > Hello everybody, > > I am trying to do particle subtraction in Relion 4.0.1 using \\ntwo mpi  > processes on a workstation with 512 Gb of RAM. I am seeing the two mpi  > proce sses \\nconsuming progressively more RAM (about 25% each in the  > attached screenshot, or 200Gb in total) \\nuntil they run out of RAM  > about half an hour into the subtraction, upon which they are killed by  > \\nthe OS. Does anybody know why this happens? I hav e never encountered  > this with Relion 3. > > \\nThank you, > > Yaroslav > > Yaroslav Tsybovsky, Ph.D. (contractor) > > Senior Scientist & Head > > Vaccine \\nResearch Center Electron Microscopy Unit > > Frederick National Laboratory for Cancer Research > > \\nLeidos Biomedical Research, Inc. > > Post Office Box B > > Frederick, Maryland 21702 > > Phone: \\n301-846-7550 > > > ------------------------------------------------------------------------  > > To unsubscribe from the \\nCCPEM list, click the following link: > \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1  > \\n<https://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1> > ', '######################################################################## To unsubscribe \\nfrom the CCPEM list, click the follo wing link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message was issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing list hosted by www.jiscmail.ac.uk, terms & conditions \\nare available at https://www.jiscmail.ac.uk/policyand security/ JISCMail - CCPEM Archives Print Hello \\neverybody, I am trying to do particle subtraction in Relion 4.0.1 using two mpi processes on a \\nworkstation with 512 Gb of RAM. I am seeing the two mpi processes consuming progressively more RAM \\n(about 25% eac h in the attached screenshot, or 200Gb in total) until they run out of RAM about half an \\nhour into the subtraction, upon which they are killed by the OS. Does anybody know why this happens? I \\nhave never encountered this with Relion 3. Thank you, Yaroslav Y aroslav Tsybovsky, Ph.D. (contractor) \\nSenior Scientist & Head Vaccine Research Center Electron Microscopy Unit Frederick National Laboratory \\nfor Cancer Research Leidos Biomedical Research, Inc. Post Office Box B Frederick, Maryland 21702 Phone: \\n301-846-7550 ######################################################################## \\nTo unsubscribe from the CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message was issued to \\nmembers of www.jiscmail.ac.u k/CCPEM, a mailing list hosted by www.jiscmail.ac.uk, terms & conditions \\nare available at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print Dear all, \\nPlease join the NIH Transformative High Resolution CryoEM Program service cent ers this week for our \\nCryoEM Current Practices webinar. Time: Thursday, June 29, 2023 at 9 AM pacific / 12 PM eastern time \\nSpeaker: Shubham Dubey & Michelle Bush, Graduate Students, Noinaj Lab, Department of Biological \\nSciences, Purdue University Title: Br eaking the iron piracy code in pathogenic Neisseria All are welcome \\nto attend. Registration is at no -cost, but sign -up is required: \\nhttps://us02web.zoom.us/webinar/register/WN_Oe0LnejKRiWvin_JEPMGMg Please forward this \\nposting to anyone that may be interes ted. Information about previous and future talks can be found at \\nhttps://www.cryoemcenters.org/events/#cryoEM -webinar Best regards, National Center for CryoEM \\nAccess and Training \\n######################################################################## To u nsubscribe \\nfrom the CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message was issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing list hosted by www.jiscmail.ac.uk, terms & conditions \\nare av ailable at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print \\nApplications are invited for a postdoctoral research associate to be based in the Department of Basic and \\nClinical Neuroscience and the Randall Centre for Cell and Mol ecular Biophysics at King s College London. \\nThe post is funded by the UK Medical Research Council for 3 years and will investigate the role of \\ndefective endoplasmic reticulum (ER) -mitochondria signalling in dementia and amyotrophic lateral \\nsclerosis (ALS).   ER-mitochondria signaling and its role in dementia/ALS are highly topical areas of \\nresearch and the grant holders have made major contributions to these fields. The successful applicant \\nwill join a multi -disciplinary group that seeks to understand the me chanisms by which ER -mitochondria \\nsignalling is perturbed in dementia/ALS and how correcting damaged ER -mitochondria signalling might \\nbe therapeutic for these major diseases.  The post will focus on gaining high -resolution structural \\ninformation on ER -mito chondria tethers using cryo -electron microscopy (cryo -EM) methods. Experience \\nin structural biology methods and especially cryo -EM, and protein purification methods will be valuable. \\nThe successful applicant will have access to state -of-the-art facilities in both King s College, the King s \\nCollege Dementia Research Institute and the Crick Institute. KCL have access to a Titan Krios with a K3 ', 'direct electron detector located at the Crick Institute and also have on -site cryo -EM sample preparation, \\nscreening a nd data collection facilities, including a JEOL CryoARM200 equipped with a K3 direct electron \\ndetector (to be installed in July 2023). You can apply and find more information here; \\nhttps://www.kcl.ac.uk/jobs/068946 -research -associate -structural -studies -of-er-mitochondria -tethers \\nYou can also contact Joe Atherton (leading the structural work, [log in to unmask]), or Chris Miller \\n(leading the overall project, [log in to unmask]) for more information. \\n########################################################### ############# To unsubscribe \\nfrom the CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message was issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing list hosted by www.jiscmail.ac.uk, terms &  conditions \\nare available at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print Dear all, \\nApplications are invited for a Postdoctoral Research Associate position, funded by the Royal Society, for \\nup to 30 months to work with Dr J on Agirre in the Department of Chemistry. The successful applicant will \\njoin a vibrant team of researchers at the heart of the world -leading York Structural Biology Laboratory \\n(YSBL). The work of the laboratory over the years has included the delivery of w orld-leading methods \\nand experimental discoveries, training and nurturing hundreds of outstanding scientists in the process. \\nRole The AlphaFold software for protein model prediction has revolutionised workflows in structural \\nbiology. However, while high -accuracy models are now available for most human proteins of medical \\nrelevance, up to 50% of those proteins are expected to be glycosylated   a modification whose \\nstructure AlphaFold does not predict. Protein glycosylation is greatly important, with implicat ions for \\nprotein folding and molecular recognition including, as evidenced by the spike glycoprotein of \\nSARS -CoV-2, viral infection. In collaboration with other scientists, we have extended the Privateer \\nsoftware to interactively build missing glycosylatio n onto precalculated AlphaFold models. In parallel, we \\nhave developed analyses that allow us to get a clear picture of glycans and their interactions, including \\nH-bonds and XH -Pi interactions. We expect that these methods will enable us to, based on existi ng \\nglycobiological knowledge, extend the Privateer software to automatically complete AlphaFold models \\nwith crucial missing glycosylation components. We are seeking a person with an interest in both biology \\nand coding, who can collaborate with others in a very diverse team with a mission to further our \\nknowledge of glycobiology and seize the opportunities presented by a deeper understanding of it. Main \\nduties and responsibilities To develop and extend the Privateer software to complete AlphaFold models \\nwith  the missing glycan components. To further our existing collaborations with CCP4, CCP -EM, \\nPDB-REDO, the Protein Data Bank and the AlphaFold Protein Structure Database. Department The \\nDepartment of Chemistry is one of the largest and most successful departm ents at York and we are \\nrenowned internationally for our research. As a department, we strive to provide a working environment \\nthat allows all staff and students to contribute fully, to flourish, and to excel. We are proud of our Athena \\nSwan Gold Award. Th e Department of Chemistry values all employees for the qualities and skills they \\nbring to the workplace and aims to be a diverse and egalitarian community in which all can thrive. For \\ninformal enquiries: please contact Dr Jon Agirre, [log in to unmask] To apply: \\nhttps://jobs.york.ac.uk/vacancy/research -associate -529961.html Deadline for applications: 20 August \\n2023. Many thanks for making it this far in the advert! With best wishes, Jon --  Dr Jon Agirre Royal \\nSociety University Research Fellow (assistant p rofessor) CCP4 WG2 co -chair | instruct -ERIC \\nrepresentative @ 3D -Bioinfo SC (Elixir) York Structural Biology Laboratory, Department of Chemistry \\nUniversity of York, Heslington, YO10 5DD, York, UK \\n############################################################# ########### To unsubscribe ', 'from the CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message was issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing list hosted by www.jiscmail.ac.uk, terms & c onditions \\nare available at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print We are \\nlooking for a PhD student, PostDoc, or Senior Scientist to join our team to support cryo -EM data \\ncollection, using conventional and novel approa ches, including diffractive imaging. The low \\nsignal -to-noise ratio in cryo -EM is limiting its performance. We explore diffractive electron imaging \\ntechnologies such as Ptychography and other 4D STEM variants to improve resolution and contrast in \\ncryo -EM. F or this, we develop novel approaches using a 300kV Titan Krios equipped with a cold -FEG, an \\nRF cavity for GHz beam pulsing, a Cs probe corrector, the usual autoloader and Falcon4i, and followed by \\nenergy filtration and super -fast hybrid pixel electron dete ction. Your profile : Candidates should have \\nexperience in electron microscopy, electron tomography and/or image processing, and be open to \\nsupport or participate in the development of new technologes at the level of hardware, EM operation, \\nand online data  processing. We offer : The EPFL ranks among the world s top scientific universities. We \\nare located in Lausanne in Switzerland, a beautiful, vibrant and highly international city on the shores of \\nthe scenic Lake Geneva, situated within an Alpine setting i n the heart of Europe. English is the main \\nlanguage at the EPFL and in our laboratory, while Lausanne is in the French -speaking part of Switzerland. \\nThe position comes with a competitive salary. EPFL is committed to being an equal opportunity employer. \\nThe successful candidate will join a vibrant, collaborative, international, and diverse research team with \\ninterdisciplinary expertise in biology, physics, computer science and engineering. The focus of research \\nat our Laboratory of Biological Electron Micros copy (LBEM) in the physics department of the EPFL is on \\nneurodegeneration and method development in cryo -EM. We enjoy unique state -of-the-art \\ninstrumentation. More information about our research group can be found at \\nhttps://lbem.ch<https://urldefense.com/ v3/__https:/lbem.ch__;!!Mih3wA!D -ORujpYZp4Fwfs_fGEKFdT0\\nE4NgeLbdVDsKqyFxzUfMYYb433cFMXTdf7yeHrc4yJLfMQc9UFt819vG5tCOwSuw3bDTAhs$> The position \\nis available immediately, funding is available, and the start date is flexible. Applications should include \\nPDF do cuments with a Cover letter, a CV, and the contact information of references. For questions or \\nadditional information, please contact Henning Stahlberg at [log in to unmask]<mailto:[log in to \\nunmask]> Henning Stahlberg Laboratory of Biological Electron Mic roscopy Institute of Physics, School of \\nBasic Sciences, EPFL, and Dep. of Fund. Microbiology, Faculty of Biology and Medicine, UNIL, Cubotron, \\nBSP421, 1015 Lausanne, Switzerland lbem.ch<https://lbem.ch/>, +41 21 693 45 07 \\n################################## ###################################### To unsubscribe \\nfrom the CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message was issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing list hosted by ww w.jiscmail.ac.uk, terms & conditions \\nare available at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print Hi, no, \\nUltraAuFoil grids do not have to be prepared, they can be used  straight out of the box. Plasmacleaning \\n- to make th em more hydrophobic  - is suggested however. Best Dario On 26.06.23 03:00, Forhad Karim \\nSaikot wrote: > Hi all, > > Apologies, but I have a quick question that suits the thread nicely.  > Should \\nwe wash UltraAuFoil grids with Ethyl Acetate or similar  > so lvent, the way we do for Quantifoil \\ngrids? > > Thanks in advance. > > > *Kind regards,* > > *Forhad Karim Saikot* > > */PhD Student/* > > \\n*Bostjan Kobe Lab* > > * > * > > School of Chemistry and Molecular Biosciences > Cooper road, St Lucia \\ncampus > The Un iversity of Queensland > Brisbane, Qld 4072, Australia. > > *E:*[log in to unmask] \\n<mailto:[log in to unmask]> > *M:*+61480106241 <tel:+61480106241> > *O:* Building 76, Room ', \"314. > > ------------------------------------------------------------------------  > *From:* Collaborative Computational \\nProject in Electron  > cryo -Microscopy <[log in to unmask]> on behalf of Daniel Asarnow  > <[log in to \\nunmask]> > *Sent:* Saturday, June 10, 2023 8:29:05 AM > *To:* [log in to unmask] <[log in to \\nunmask]> > *Subject:* Re: [ccpem] UltraAuFoil grid images has dark background > Hi Celine, > The white \\nspots are reflections from the gold. They can be ameliorated  > to some extent with an objective \\naperture or a smaller beam, but in  > practice they don't seem to be a big pro blem, so I usually \\ndon't  > bother. If there are gold edges in some of the images, there can be  > strong low -frequency \\neffects that may hurt patch -wise CTF fits (as in  > cryoSPARC). In that case you can try using a higher \\nminimum resolution  > during CTF  estimation (like 20  ), or run the images with bad fits  > through \\nCTFFind4 to try and recover those. > > With respect to the particles/ice in the holes, you can pay \\nattention  > to the visible size of the squares and the appearance of holes at the  > hol e / SA view \\nmagnification, and try to figure out which ones are  > likely to have good space for imaging. If there \\naren't enough of those  > good holes, then sample prep can be adjusted to maximize those areas.  > \\nThis is all just the same advice as for ho ley carbon or other types of  > grids, but for Ultraufoils \\nspecifically I recommend using a lower  > concentration than holey carbon (probably about half), using \\na large  > enough droplet size (3 -3.5  L), and manually spreading the droplet  > over the grid  while \\npipetting the sample on. If you look closely at  > the drop it should be a dome with a flat bottom, not a \\nbead with round  > edges where it contacts the grid. > > Best, > -da > > On Fri, Jun 9, 2023 at 1:38 PM \\nShi Celine <[log in to unmask]> wrote: > >     Hi all, I frozen my protein using UltrAuFoil R1. 2/1. 3, 300 \\nmesh >     gold grids. Only in a small protion of the holes, I can see good >     particles with clear \\nbackgound. Most of the images have dark >     background and white spots see attachm ents. I did >     \\nZjQcmQRYFpfptBannerStart >     This Message Is From an Untrusted Sender >     You have not \\npreviously corresponded with this sender. >     ZjQcmQRYFpfptBannerEnd >     Hi all, > >     I \\nfrozen my protein using UltrAuFoil R1.2/1.3, 300 mes h gold >     grids. Only in a small protion of the \\nholes, I can see good >     particles with clear backgound. Most of the images have dark >     \\nbackground and white spots see attachments. I did plasma cleaning >     for 30s or 1 min using H2, O2. \\nThe atl as of the grids seems good. > >      Any suggestions? Thank in advance for taking the time to \\nread >     this email. > >     Best, >     Celine > >     \\n------------------------------------------------------------------------  > >     To unsubscribe from the  CCPEM list, \\nclick the following link: >     https://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 >     \\n<https://urldefense.com/v3/__https://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1__\\n;!!LQC6Cpwp!plQYbp2kECvBJ2y2GzqJLPX_TCJdH_bwmdq MbfyIjox23KZdfXoepMBmSu93mkyLeg77Ca_qP\\nBRLdVFSsTxuyJZL$> > > > > ------------------------------------------------------------------------  > > To unsubscribe \\nfrom the CCPEM list, click the following link: > \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBE D1=CCPEM&A=1  > \\n<https://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1> > > > \\n------------------------------------------------------------------------  > > To unsubscribe from the CCPEM list, click the \\nfollowing link: > https://www.jiscmail.ac.uk/ cgi-bin/WA -JISC.exe?SUBED1=CCPEM&A=1  > \\n<https://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1> > \\n######################################################################## To unsubscribe \\nfrom the CCPEM list, click the following link: \\nhttps://www.j iscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message was issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing list hosted by www.jiscmail.ac.uk, terms & conditions \\nare available at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print Dear K3 \", \"owners, if anyone on this list wants/needs to sell a (faulty) K3 server, e.g.  after a switch to a different \\ndetector, I'd be very happy to hear back  from you. The MPI of Biophysics in Frankfurt, Germany, is \\nlooking to use  one f or spare parts. My contact address is: [log in to unmask] With best regards from \\nFrankfurt, Sonja --  Sonja Welsch Head of Central Electron Microscopy Facility Max Planck Institute of \\nBiophysics Max -von-Laue -Strasse 3 60438 Frankfurt am Main, Germany Email : [log in to unmask] Mobile: \\n+49 157 80625946 Phone: + 49 69 6303 3015 \\n######################################################################## To unsubscribe \\nfrom the CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUB ED1=CCPEM&A=1 This message was issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing list hosted by www.jiscmail.ac.uk, terms & conditions \\nare available at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print You are \\ncordially i nvited to join  .. Dominique Bourgeois Institut de Biologie Structurale J -P . Ebel, GRENOBLE \\nWednesday, June 28, 2023, 13:30 EDT (New York time) STRUCTURAL PHOTOPHYSICS OF FLUORESCENT \\nPROTEINS USED IN SUPER RESOLUTION MICROSCOPY \\nhttps://bnl.zoomgov.com/meet ing/register/vJIsf -2orTkoHue9RhbBuN5K0UKRrafNNTk Abstract \\nPhototransformable fluorescent proteins (PTFPs) are widely used in many advanced fluorescence \\nmicroscopy methods, notably in the super -resolution field. A plethora of PTFPs have been engineered in \\nthe last years, which display a variety of photoactivation, photoconversion, photoswitching, \\nphotoblinking and photobleaching properties that can be used at advantage in many super -resolution \\nschemes, but that are also at the origin of significant complicat ions and artifacts, making these \\nfascinating genetically encoded labels still far from ideal. In this talk, I will review how (kinetic) X -ray \\ncrystallography, optical spectroscopy, multidimensional NMR and molecular dynamics simulations can be \\ncombined wit h single -molecule investigations to reveal phototransformation mechanisms of PTFPs at the \\nnear -atomic level.  Graphical Abstract: [cid:653945e9 -d41c -4f3d -bb0a -2b57aba4efed] \\n=============================================================================== Viv ian \\nStojanoff, PhD Education, Training, Outreach User Program p 1(631) 344 8375 e [log in to unmask] w \\nhttps://www.bnl.gov/ps/lifesciences/<https://www.bnl.gov/ps/lsbr/> Address: Center for Biomolecular \\nStructure National Synchrotron Light Source II Buildi ng 745 Brookhaven National Laboratory Upton NY \\n11973 Supporting Grants: CBMS is supported by NIH -NIGMS #P30GM133893, and by the DOE -BER \\n#KP1605010. Any work performed at NSLS -II is supported by DOE -BES  under contract # DE -SC0012704. \\n###################### ################################################## To unsubscribe \\nfrom the CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message was issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing list hosted by www.jiscmail.ac.uk, terms & conditions \\nare available at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print Dear all, \\nWe would like to estimate the number of Spotiton time -resolved cryoEM projects that require mixing of \\ntwo samples from 100 -500 ms. Spotiton is a unique, robotic grid preparation device able to dispense 2 \\nsamples independently that mix for 100 -500 ms on a plunging nanowire -coated grid prior to vitrification, \\npotentially enabling the capture of short -lived, structural intermediates. Currently, all grids are prepared \\nat room temperature.  See Dandey et al, 2020 (PMID: \\n32778833)<https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7799389/> and Budell et al, 2021 (PMID: \\n33720116)<https://www.jove.com/t/62271/cryo -elect ron-microscopic -grid-preparation -for-time -resolve\\nd-studies> for a description of the device and method. Please contact [log in to unmask]<mailto:[log in \\nto unmask]> if Spotiton could aid your research or if you have questions. Please include a short \", 'paragr aph briefly stating the project, the protein mass, and timescales of interest. Please forward this \\nemail to any colleagues who may have applicable projects. Note that there are no geographical \\nrestrictions and Spotiton is limited to BSL -1 projects. Best re gards, Alex Noble Simons Resource for \\nAutomated Molecular Microscopy (SRAMM) Simons Electron Microscopy Center (SEMC) New York \\nStructural Biology Center (NYSBC) \\n######################################################################## To unsubscribe \\nfrom th e CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message was issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing list hosted by www.jiscmail.ac.uk, terms & conditions \\nare available at https:/ /www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print Dear \\ncryo -EM and cryo -ET community, I\\'d like to draw your attention to a postdoctoral position available in  \\nour Microscopic Imaging of Complex Assemblies (MICA) group at the  Institute  of Structural Biology \\n(IBS), Grenoble, France. We have several  promising projects underway on microbial and mitochondrial \\nmolecular  machines, and are looking for an enthusiastic candidate with experience  in the \\ncomputational aspects of cryo -EM and cryo -ET. If you are  motivated to apply current software and go \\nbeyond the state of the art  by developing your own tools to solve and analyse structures and  \\ndynamics of flexible, heterogeneous samples and their supramolecular  organisation in increasingly \\ncomplex environments, please email me at  [log in to unmask] Knowledge and experience in Python \\nand deep  learning frameworks would be an advantage. For those who love or wish to  get trained in \\nexperimental cryo -EM, access to the excellent cryo -EM  platform  of the IBS is available. Otherwise, \\ncryo -ET and single particle  cryo -EM data sets on several challenging projects have already been  \\ncollected and are waiting for you to join us in unlocking their  mysteries. This is a two -year, renewable \\nposition offeri ng a unique  opportunity to work on exciting projects within a small and friendly  \\ninterdisciplinary research group, within a top research environment in  the French Alps. We are looking \\nforward to hearing from you, Irina Gutsche, MICA group, IBS Grenoble \\n######################################################################## To unsubscribe \\nfrom the CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message was issued to \\nmembers of www.jiscmail.ac.uk/ CCPEM, a mailing list hosted by www.jiscmail.ac.uk, terms & conditions \\nare available at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print --------  \\nMensaje reenviado --------  Asunto:  AlgoSB 2023: FROM STRUCTURE RESOLUTION TO DYN AMICAL \\nMODELING IN  CRYO -ELECTRON MICROSCOPY Fecha:  Mon, 26 Jun 2023 09:39:21 +0200 (CEST) De:  \\n[log in to unmask] Responder a:  [log in to unmask] Para:  [log in to unmask] Dear Colleague, Please \\nfind below the announcement for the 9th edition of the Win ter School on Algorithms in Structural \\nBioinformatics (AlgoSB). This year\\'s topic is FROM STRUCTURE RESOLUTION TO DYNAMICAL MODELING \\nIN CRYO -ELECTRON MICROSCOPY The school has received the CNRS label \"Ecole Th matique\", and is \\nsponsored by the CNRS, Inria,  IESC, and GDR Bioinformatique Moleculaire. We would appreciate your \\nforwarding it to PhD students, postdocs, or other colleagues who might be interested. Best regards, F. \\nCazals, on behalf of the organization and scientific committees https://project.inri a.fr/algosb2023/ --- \\n___________________________________________________________________________ WINTER \\nSCHOOL, ALGORITHMS IN STRUCTURAL BIOINFORMATICS : FROM STRUCTURE RESOLUTION TO \\nDYNAMICAL MODELING IN CRYO -ELECTRON MICROSCOPY https://project.inria.fr/a lgosb2023/ \\nNovember 19 -24, 2023 Institute of Scientific Study of Carg se, https://iesc.universita.corsica/ \\n___________________________________________________________________________ We are ', 'pleased to announce the 9th edition of the Algorithms in Structura l Bioinformatics (AlgoSB) winter \\nschool. The aim of AlgoSB is to introduce advanced methods in Structural Bioinformatics in the largest \\nsense, giving special attention to interdisciplinary approaches. This year\\'s focus will be on modeling in \\ncryo -electron  microscopy. PROGRAM Each day\\'s course consists of a lecture in the morning followed by \\na practical session on the participants\\' computers in the afternoon. The topics of the classes are the \\nfollowing (details at https://project.inria.fr/algosb2023/lecture s/) I The image alignment problem Carlos \\nOscar Sozano   National Center of Biotechnology, Madrid, Spain II From raw data to context Christian \\nDienemann   MPI Gottingen, Germany III Combining molecular dynamics and cryo -EM Pilar Cossio   \\nFlatiron Institute,  New -York, USA IV Flexibility in cryo -EM reconstruction Johannes Schwab   Cambridge \\nUniv, England V Hybrid methods for analyzing conformational variability in cryo -EM and  cryo -Et data \\nSlavica Jonic   CNRS, France VENUE The school will take place at Instit ut d\\' tudes Scientifiques de Carg \\nse - Universit  de Corse (https://iesc.universita.corsica/), Corsica. AUDIENCE The AlgoSB school is \\ntargeted towards PhD students, postdocs, and junior researchers from public research institutions and \\nindustry, both in Fr ance and internationally. This school is limited to 35 participants. It is expected that all \\nwill participate for the duration of the school. APPLICATION A one -page application (pdf format) should \\nbe sent by e -mail to [log in to unmask] with the Subject \"A lgoSB application . The application should \\nprovide a mini -vitae or a pointer to the full vitae, a brief presentation of the applicant s research \\ninterests, and a description of how the offered courses can benefit their work. Institute for Scientific \\nStudy of Cargese (IESC) Application deadline: July 28, 2023 Confirmation of participation:mid August \\nRegistration: late August 2023 Registrations fees (*): - free for CNRS agents (NB: the school is a CNRS \\nthematic school), - 1000 euros regular rate (non -academic ), - 600 euros for researchers and professors \\nfrom Public Research  Institutions and Universities, - 450 euros for post -doctoral students, - 350 euros \\nfor PhD students. The registration fee includes housing at the IESC, lunches and coffee  breaks. * \\nRegist ration fees may be reduced if more sponsors are identified, and will be waived if the school is \\nrequired to take place virtually. ORGANIZERS Organization Committee Fr d ric Cazals, Inria \\nSophia -Antipolis -M diterran e Isaure Chauvot -de-Beauch ne, LORIA -CNRS  Allison Ballandras -Colas, \\nIBS-CNRS, Grenoble Scientific Committee Fr d ric Cazals, Inria Sophia -Antipolis -M diterran e Isaure \\nChauvot -de-Beauch ne, LORIA -CNRS Juan Cort s, LAAS -CNRS Yann Ponty, Ecole Polytechnique Charles H. \\nRobert, LTB -CNRS \\n############# ########################################################### To unsubscribe \\nfrom the CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message was issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mail ing list hosted by www.jiscmail.ac.uk, terms & conditions \\nare available at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print *Your \\nmission** :* Developing features for the PSI Data Catalog as part of the Open EM Data Network gra nt. \\nBuild up the pipe - line for data exchange between the DCI -Lausanne (UNIL/EPFL) and the Open EM Data \\nNetwork. *Function*: Software Engineer *Duration*: 2.5 years *Main tasks*: * Developing features for \\nthe PSI Data Catalog as part of the Open EM Data Ne twork grant * Contributing to data catalog \\ndevelopment, training, and operations * Developing workflows and APIs to integrate PSI & Swiss \\nelectron microscopy facilities with PSI systems * Extending the toolset for data management and \\nmetadata harvesting * Advising the scientists in integration tasks related to data curation for their \\ninfrastructure * Representing the needs of PSI/AWI related to the topic of data curation within national \\nand international projects * Applying and conceptualizing standards and  ontologies for EM metadata * \\nProviding technical documentation & user training *Here is the link for the job application*: ', \"https://recruiting.epfl.ch/SelfServiceLine/Vacancies/2973/SelectionOE or, in case of a problem during \\nthe application let me know by  email - [log in to unmask] \\n######################################################################## To unsubscribe \\nfrom the CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message was issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing list hosted by www.jiscmail.ac.uk, terms & conditions \\nare available at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print Hi all, \\nApologies, but I have a quick question that suits the thread nicely. Should we wash UltraAuFoil grids with \\nEthyl Acetate or similar solvent, the way we do for Quantifoil grids? Thanks in advance. Kind regards, \\nForhad Karim Saikot PhD Student Bostjan Kobe Lab School of Chemistry and Molecular Biosciences \\nCoope r road, St Lucia campus The University of Queensland Brisbane, Qld 4072, Australia. E: [log in to \\nunmask]<mailto:[log in to unmask]> M: +61480106241<tel:+61480106241> O: Building 76, Room 314. \\n________________________________ From: Collaborative Computatio nal Project in Electron \\ncryo -Microscopy <[log in to unmask]> on behalf of Daniel Asarnow <[log in to unmask]> Sent: Saturday, \\nJune 10, 2023 8:29:05 AM To: [log in to unmask] <[log in to unmask]> Subject: Re: [ccpem] UltraAuFoil \\ngrid images has dark backgro und Hi Celine, The white spots are reflections from the gold. They can be \\nameliorated to some extent with an objective aperture or a smaller beam, but in practice they don't \\nseem to be a big problem, so I usually don't bother. If there are gold edges in so me of the images, there \\ncan be strong low -frequency effects that may hurt patch -wise CTF fits (as in cryoSPARC). In that case you \\ncan try using a higher minimum resolution during CTF estimation (like 20  ), or run the images with bad \\nfits through CTFFind4 to try and recover those. With respect to the particles/ice in the holes, you can pay \\nattention to the visible size of the squares and the appearance of holes at the hole / SA view \\nmagnification, and try to figure out which ones are likely to have good spa ce for imaging. If there aren't \\nenough of those good holes, then sample prep can be adjusted to maximize those areas. This is all just \\nthe same advice as for holey carbon or other types of grids, but for Ultraufoils specifically I recommend \\nusing a lower c oncentration than holey carbon (probably about half), using a large enough droplet size \\n(3-3.5  L), and manually spreading the droplet over the grid while pipetting the sample on. If you look \\nclosely at the drop it should be a dome with a flat bottom, not a bead with round edges where it \\ncontacts the grid. Best, -da On Fri, Jun 9, 2023 at 1:38 PM Shi Celine <[log in to unmask]<mailto:[log in to \\nunmask]>> wrote: Hi all, I frozen my protein using UltrAuFoil R1. 2/1. 3, 300 mesh gold grids. Only in a \\nsmall pro tion of the holes, I can see good particles with clear backgound. Most of the images have dark \\nbackground and white spots see attachments. I did ZjQcmQRYFpfptBannerStart This Message Is From an \\nUntrusted Sender You have not previously corresponded with thi s sender. ZjQcmQRYFpfptBannerEnd Hi \\nall, I frozen my protein using UltrAuFoil R1.2/1.3, 300 mesh gold grids. Only in a small protion of the \\nholes, I can see good particles with clear backgound. Most of the images have dark background and \\nwhite spots see at tachments. I did plasma cleaning for 30s or 1 min using H2, O2. The atlas of the grids \\nseems good.  Any suggestions? Thank in advance for taking the time to read this email. Best, Celine \\n________________________________ To unsubscribe from the CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1<https://urldefense.com/v3/__htt\\nps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1__;!!LQC6Cpwp!plQYbp2kECvBJ2y2G\\nzqJLPX_TCJdH_bwmdqMbfyIjox23KZdfXoepMBmSu93m kyLeg77Ca_qPBRLdVFSsTxuyJZL$> \\n________________________________ To unsubscribe from the CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 \\n################################################################### ##### To unsubscribe \", 'from the CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message was issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing list hosted by www.jiscmail.ac.uk, terms & conditi ons \\nare available at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print Hi,  > 1) I \\nhave Falcon 4 EER data. The docs  >  \\n<https://relion.readthedocs.io/en/latest/Reference/MovieCompression.html>  recommend 4K \\nrendering, so that\\'s  what I want to do. For motion  correction I\\'ve used a --dose_per_frame of 1.5, \\nwhile it should have  been 2.0. Could that be a problem? Using a too high number means \\ndown -weighting high resolution signals too much. But I don\\'t think it causes such a huge deterioration \\nfrom 2.3 A to 12 A.  > The docs say: \"You can change this later.\", where and how can I  change > this \\nlater? Polishing can change this but in your case it won\\'t work since the resolution is just too bad. You \\nhave to repeat motion correction.  > 2) How do I render in a 8k grid in Relion? Hypothetically, I don\\'t  > \\nactually want to do that. But it says in the docs: \"|Binning  > factor| should be 2 to bring a 8K super \\nresolution grid into a 4K\". When  > I set the binning factor to 2, I get 2k ima ges. With 1, I get 4k \\nimages.  > Am I correct to put binning and eer_upsampling to 1? --eer_upsampling 2 does 8K rendering.   \\nbinning factor 1 gives you 8K images, while 2 gives 4K. --eer_upsampling 1 (default) does 4K rendering.   \\nbinning factor 1 gives y ou 4K images, while 2 gives 2K. Best regards, Takanori Nakane On 2023/06/26 \\n3:27, Bastiaan Bergman wrote: > Dear CCPEM -ers, >  > I have a dataset with successful (2.3A) Cryospark \\nreconstruction but  > don\\'t seem to get it working with Relion. I\\'m sure I\\'m doing something  > wrong, \\nbut need some help identifying what that is. The best I got so  > far is 12A. A few things I\\'d like to check \\nwith the community: >  > 1) I have Falcon 4 EER data. The docs  > \\n<https://relion.readthedocs.io/en/latest/Reference/Movi eCompression.html> recommend 4K rendering, \\nso that\\'s what I want to do. For motion correction I\\'ve used a --dose_per_frame of 1.5, while it should \\nhave been 2.0. Could that be a problem? The docs say: \"You can change this later.\", where and how can I \\nchang e this later? >  > 2) How do I render in a 8k grid in Relion? Hypothetically, I don\\'t  > actually want \\nto do that. But it says in the docs: \"|Binning  > factor| should be 2 to bring a 8K super resolution grid \\ninto a 4K\". When  > I set the binning factor to  2, I get 2k images. With 1, I get 4k images.  > Am I \\ncorrect to put binning and eer_upsampling to 1? >  > Thanks for your help here! >  > Cheers, \\nBastiaan. > Model Medicines <http://www.modelmedicines.com> >  >  >  > \\n-------------------------------------- ----------------------------------  >  > To unsubscribe from the CCPEM list, click \\nthe following link: > https://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1  > \\n<https://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1> >  \\n################ ######################################################## To unsubscribe \\nfrom the CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message was issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing  list hosted by www.jiscmail.ac.uk, terms & conditions \\nare available at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print Dear \\nCCPEM -ers, I have a dataset with successful (2.3A) Cryospark reconstruction but don\\'t seem to get it \\nworking with Relion. I\\'m sure I\\'m doing something wrong, but need some help identifying what that is. \\nThe best I got so far is 12A. A few things I\\'d like to check with the community: 1) I have Falcon 4 EER data. \\nThe docs <https://relion.readthedocs.io/en/l atest/Reference/MovieCompression.html> recommend 4K \\nrendering, so that\\'s what I want to do. For motion correction I\\'ve used a --dose_per_frame of 1.5, while \\nit should have been 2.0. Could that be a problem? The docs say: \"You can change this later.\", where  and \\nhow can I change this later? 2) How do I render in a 8k grid in Relion? Hypothetically, I don\\'t actually ', 'want to do that. But it says in the docs: \"Binning factor should be 2 to bring a 8K super resolution grid \\ninto a 4K\". When I set the binning facto r to 2, I get 2k images. With 1, I get 4k images. Am I correct to put \\nbinning and eer_upsampling to 1? Thanks for your help here! Cheers, Bastiaan. Model Medicines \\n<http://www.modelmedicines.com> \\n############################################################ ############ To unsubscribe \\nfrom the CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message was issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing list hosted by www.jiscmail.ac.uk, terms & conditions \\nare available at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print Hello EM \\ncolleagues, We are facing an interesting situation with 3D classification without alignment in RELION. We \\nhave a dataset of ~1 million partic les from ~18,000 micrographs that we processed in cryosparc all the \\nway to heterogenous refinement, which showed several distinct conformational states. We then \\nperformed their individual homogenous refinements, which yielded nice detailed maps showing \\nconformational differences between these states. As part of our pipeline, we performed a consensus \\nhomogenous refinement of the particles in cryosparc, and then exported the particles and the volume to \\nRELION for 3D classification. We run into two issues - 1. RELION complains that there are very few \\nparticles per group and that we should consider merging micrographs together. 2. The output from this \\n3D classification fails to separate the states. In fact, if I use one of the different states (low -passed to 20A)  \\nas the volume in 3D classification without alignment, I see it morph into the consensus structure as we \\nprogress through the iterations. My questions is: are these two problems related? To overcome this, \\nshould we run one iteration of 3D classification, t hen use the newly generated model.star file in subset \\nselection to reduce the number of groups, and now use this new star file in a fresh 3D classification \\nwithout alignment? Would that be the right approach? All and any advise will be much appreciated. \\nThank you. Saif S. Saif Hasan, PhD Assistant Professor Department of Biochemistry and Molecular Biology, \\nUniversity of Maryland School of Medicine 108 N. Greene Street, Baltimore MD 21201 \\nhttps://www.medschool.umaryland.edu/profiles/Hasan -S-Saif-Hasan -Syed -Saif/ \\n######################################################################## To unsubscribe \\nfrom the CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message was issued to \\nmembers of www.jiscmail.a c.uk/CCPEM, a mailing list hosted by www.jiscmail.ac.uk, terms & conditions \\nare available at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print Hi all - \\nthe solution we ve come up at this he CCR is to have a server instance per P I group and have all members \\nof the group use the single instance. This then means we limit the number of independent servers we \\nneed to manage while keeping track of usage at the group level (which for us is sufficient granularity) All \\nthe best Susan Sent  from my iPhone On Jun 22, 2023, at 7:51 PM, Rick Baker <[log in to unmask]> wrote: \\nI would like to  second  this discussion point. At UNC we are currently trying to implement cryoSPARC \\non our HPC and we are running into similar issues. Workarounds are wel come, but also we would like to \\ncommunicate to the Structura Bio team that this is a feature many groups would be Interested in. \\nParadoxically cryoSPARC was the first  workstation friendly  cryoEM processing software, but now we \\nall want cryoSPARC integrat ed into our shared clusters. -Rick Baker On Thu, Jun 22, 2023 at 6:18 AM \\nPaolo Swuec <[log in to unmask]<mailto:[log in to unmask]>> wrote: Dear All, We are having a hard time \\nhandling cryoSPARC on our IT infrastructure and we would love to hear your feedb ack and advice. In its \\ncurrent implementation, cryoSPARC runs under one master account (cryosparc_user), which is shared by \\neverybody logged into cryoSPARC. All files are therefore written and managed by the master account, ', 'posing a serious issue with data  tracking and accountability. Being a public funded institute, at Human \\nTechnopole we comply to European ISO certifications, and we need to guarantee and maintain data \\ntracking on our IT infrastructure. The solution we have currently adopted is putting the  data to be \\nprocessed by cryoSPARC on a segregated server, with the inconvenience of going back and forth between \\nthat  quarantine area  and the main storage. Still, it would be nice to easily hop from Relion to CS \\nwithin the same directory. We are really eager to hear from other solutions adopted from any other \\nresearch institute or even companies. Best, Paolo             Paolo Swuec Head of Cryo EM Facility \\nCryo EM Facility Human Technopole Palazzo Italia \\nViale<https://www.google.com/maps/search/Viale+Rit a+Levi+Montalcini,+1+20157+Milan,+Italy?entry=\\ngmail&source=g> \\nRita<https://www.google.com/maps/search/Viale+Rita+Levi+Montalcini,+1+20157+Milan,+Italy?entry=g\\nmail&source=g> \\nLevi<https://www.google.com/maps/search/Viale+Rita+Levi+Montalcini,+1+20157+Milan,+ Italy?entry=g\\nmail&source=g> \\nMontalcini,<https://www.google.com/maps/search/Viale+Rita+Levi+Montalcini,+1+20157+Milan,+Italy?\\nentry=gmail&source=g> \\n1<https://www.google.com/maps/search/Viale+Rita+Levi+Montalcini,+1+20157+Milan,+Italy?entry=gm\\nail&source=g> \\n20157<https://www.google.com/maps/search/Viale+Rita+Levi+Montalcini,+1+20157+Milan,+Italy?entry\\n=gmail&source=g> \\nMilan,<https://www.google.com/maps/search/Viale+Rita+Levi+Montalcini,+1+20157+Milan,+Italy?entry\\n=gmail&source=g> \\nItaly<https://www.google.com/maps /search/Viale+Rita+Levi+Montalcini,+1+20157+Milan,+Italy?entry=\\ngmail&source=g> T. +39 02 -30247442<tel:02 -30247442> M. +39 3427758814<tel:3427758814> E. [log in \\nto unmask]<mailto:[log in to unmask]> <https://www.facebook.com/htechnopole> <image647418.png> \\n<https://twitter.com/humantechnopole> <image609817.png> \\n<https://www.linkedin.com/company/htechnopole> <image850345.png> \\n<https://www.instagram.com/humantechnopole/> <image641637.png> <image538247.jpg> Prima di \\nstampare questa e -mail, pensa all ambiente. Be fore printing this e -mail, think of the environment Legal \\nDisclaimer The information contained in or accompanying this email and any attachment thereto, is \\nintended solely for the use of the stated recipient(s) and may contain information that is confident ial \\nand/or privileged. Any dissemination, distribution or copying of this email and any attachment by anyone \\nwho is not the stated recipient is strictly prohibited. If you receive this message in error, please notify the \\nsender immediately and delete the m essage and any attachment from your system without retaining a \\ncopy. ________________________________ To unsubscribe from the CCPEM list, click the following link: \\nBad URL Removed - see why - \\nhttps://ees.sps.nih.gov/services/Pages/Anti -Virus.aspx?SUBED1=CC PEM&A=1<Bad%20URL%20Removed\\n%20 -%20see%20why%20 -%20https://ees.sps.nih.gov/services/Pages/Anti -Virus.aspx?SUBED1=CCPEM&\\nA=1> ________________________________ To unsubscribe from the CCPEM list, click the following link: \\nBad URL Removed - see why - \\nhttps://ee s.sps.nih.gov/services/Pages/Anti -Virus.aspx?SUBED1=CCPEM&A=1<Bad%20URL%20Removed\\n%20 -%20see%20why%20 -%20https://ees.sps.nih.gov/services/Pages/Anti -Virus.aspx?SUBED1=CCPEM&\\nA=1> CAUTION: This email originated from outside of the organization. Do not click l inks or open \\nattachments unless you recognize the sender and are confident the content is safe. ', '######################################################################## To unsubscribe \\nfrom the CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.u k/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message was issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing list hosted by www.jiscmail.ac.uk, terms & conditions \\nare available at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print I would \\nlike to  second  this discussion point. At UNC we are currently trying to implement cryoSPARC on our \\nHPC and we are running into similar issues. Workarounds are welcome, but also we would like to \\ncommunicate to the Structura Bio team that thi s is a feature many groups would be Interested in. \\nParadoxically cryoSPARC was the first  workstation friendly cryoEM processing software, but now we all \\nwant cryoSPARC integrated into our shared clusters. -Rick Baker On Thu, Jun 22, 2023 at 6:18 AM Paolo \\nSwuec <[log in to unmask]> wrote: > Dear All, > > > > We are having a hard time handling cryoSPARC on \\nour IT infrastructure and > we would love to hear your feedback and advice. In its current > \\nimplementation, cryoSPARC runs under one master account (cryo sparc_user), > which is shared by \\neverybody logged into cryoSPARC. All files are therefore > written and managed by the master account, \\nposing a serious issue with data > tracking and accountability. Being a public funded institute, at \\nHuman > Technopole w e comply to European ISO certifications, and we need to > guarantee and \\nmaintain data tracking on our IT infrastructure. The solution > we have currently adopted is putting the \\ndata to be processed by cryoSPARC > on a segregated server, with the inconvenie nce of going back and \\nforth > between that  quarantine area  and the main storage. > > > > Still, it would be nice to easily \\nhop from Relion to CS within the same > directory. We are really eager to hear from other solutions \\nadopted from > any other resear ch institute or even companies. > > > > Best, > > > > \\nPaolo > > >             > Paolo Swuec > Head of Cryo EM Facility > Cryo EM Facility > Human \\nTechnopole > Palazzo Italia > Viale > \\n<https://www.google.com/maps/search/Viale+Rita+Levi+Montalcini,+1+20157+ Milan,+Italy?entry=gmai\\nl&source=g> >  Rita > \\n<https://www.google.com/maps/search/Viale+Rita+Levi+Montalcini,+1+20157+Milan,+Italy?entry=gmai\\nl&source=g> >  Levi > \\n<https://www.google.com/maps/search/Viale+Rita+Levi+Montalcini,+1+20157+Milan,+Italy?entry=gma i\\nl&source=g> >  Montalcini, > \\n<https://www.google.com/maps/search/Viale+Rita+Levi+Montalcini,+1+20157+Milan,+Italy?entry=gmai\\nl&source=g> >  1 > \\n<https://www.google.com/maps/search/Viale+Rita+Levi+Montalcini,+1+20157+Milan,+Italy?entry=gmai\\nl&source=g> > 201 57 > \\n<https://www.google.com/maps/search/Viale+Rita+Levi+Montalcini,+1+20157+Milan,+Italy?entry=gmai\\nl&source=g> >  Milan, > \\n<https://www.google.com/maps/search/Viale+Rita+Levi+Montalcini,+1+20157+Milan,+Italy?entry=gmai\\nl&source=g> >  Italy > \\n<https://www.g oogle.com/maps/search/Viale+Rita+Levi+Montalcini,+1+20157+Milan,+Italy?entry=gmai\\nl&source=g> > T. +39 02 -30247442 > M. +39 3427758814 > E. [log in to unmask] > > \\n<https://www.facebook.com/htechnopole> > <https://twitter.com/humantechnopole> > \\n<https://www. linkedin.com/company/htechnopole> > \\n<https://www.instagram.com/humantechnopole/> > Prima di stampare questa e -mail, pensa all \\nambiente. > Before printing this e -mail, think of the environment > Legal Disclaimer > The information \\ncontained in or accompanyin g this email and any attachment > thereto, is intended solely for the use of ', 'the stated recipient(s) and may > contain information that is confidential and/or privileged. Any > \\ndissemination, distribution or copying of this email and any attachment by > an yone who is not the \\nstated recipient is strictly prohibited. If you > receive this message in error, please notify the sender \\nimmediately and > delete the message and any attachment from your system without retaining a > \\ncopy. > > ------------------------- ----- > > To unsubscribe from the CCPEM list, click the following link: > \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 > \\n######################################################################## To unsubscribe \\nfrom the CCPEM list, click t he following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message was issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing list hosted by www.jiscmail.ac.uk, terms & conditions \\nare available at https://www.jiscmail.ac.uk/p olicyandsecurity/ JISCMail - CCPEM Archives Print The Du \\nLab and the L  Lab at Van Andel Institute are dedicated to unraveling the intricate molecular \\nmechanisms underlying the body s perception of sensory information, such as taste, smell, vision, and \\ntemperature. Our mission is to investigate how signals are transmitted to the brain and ultimately \\ngenerate responses. We specifically focus on studying the fundamental ion channels/receptors \\nassociated with these senses. To achieve our research goals, we emp loy a diverse range of cutting -edge \\nstructural and biophysical techniques, including single -particle cryo -EM, patch -clamp electrophysiology, \\nand various biophysical methods.  At our state -of-the-art cryo -EM facility in the Van Andel Institute, we \\nhave adva nced equipment to facilitate our research, including a Titan Krios microscope equipped with a \\nK3 camera, as well as a Talos Arctica microscope equipped with a K2 camera (to be upgraded to K3 this \\nyear).   For further information about our labs and ongoing projects, please visit our lab website: \\nhttps://dululabs.com. Postdoctoral Fellows at VAI are full -time employees and receive an annual \\ncompetitive salary (starting salaries $70,000 per year). Qualifications Preferred, but not required: \\nExperience in struc tural biology, membrane protein biochemistry, or ion channel electrophysiology. If \\nyou are interested in joining, please send your CV, cover letter, and a list of 2 -3 referees to Juan Du ([log \\nin to unmask]) or Wei L  ([log in to unmask]). Van Andel Instit ute (VAI) provides postdoctoral training \\nopportunities to scientists beginning their research careers. The fellowships are to help promising \\nscientists advance their knowledge and research experience and at the same time support the research \\nendeavors of V AI under the mentorship of a VAI Faculty member.  While this project is currently funded, \\nan integral part of postdoctoral training is the process of preparing a research proposal and writing a \\nfellowship application. All VAI Postdoctoral Fellows are encou raged to apply for external funding as soon \\nas it is practical and reasonable after hire. Receiving a fellowship award provides recognition for both the \\nfellow and VAI! Postdoctoral Fellows at VAI receive benefits including: Medical, dental and vision \\ncove rage  Employer -sponsored life and AD&D insurance  Additional voluntary life and AD&D insurance \\nfor employees and dependents  Short -term and long -term disability insurance  Flexible -spending \\naccounts for health and child/elder care  401 K retirement savings  plans with employee/employer \\ncontributions  Paid vacation, holidays, personal days  Relocation Assistance If you have questions \\nabout the application process or working at VAI you are welcome to email Megan Doerr, Talent \\nAcquisition Specialist at [log in to unmask]    VAI is committed to achieving excellence through \\nincreasing diversity. We actively encourage applications from women, persons of color, persons with \\ndisabilities, and other underrepresented groups. Van Andel Institute is an Equal Employment \\nOpportunity (EEO) and Vietnam Era Veterans Readjustment Assistance Act (VEVRAA) Employer.  As Van \\nAndel Institute is a drug and alcohol -free employer, all prospective new employees undergo a \\nurinalysis/drug test as part of our pre -employment process.  This is a 7 -panel drug screen and it ', 'includes testing for Marijuana, Cocaine, Amphetamines, Opiates, PCP , Barbiturates, and Benzodiazepines. \\nAbout Grand Rapids, MI Van Andel Institute is located in downtown Grand Rapids, MI which is the \\nsecond largest city in M ichigan and the largest city in West Michigan. Grand Rapids offers a remarkable \\ncombination of cosmopolitan atmosphere and small town warmth, known for its philanthropic and \\nsustainability -driven community, and is ranked one of \"America\\'s Top 100 Places to  Live.\" Located on the \\nbanks of the Grand River, Grand Rapids offers all the big -city excitement you\\'d expect from a \\nmillion -resident metropolis, featuring hundreds of restaurants and nightspots, theatres, museums, \\nsports and concerts. The safe and clean d owntown is surrounded by an eclectic mix of walk -able \\nneighborhoods.  With a medical and life sciences industry boom, strong entrepreneurial activity, \\ndedication to green building and living, and innovative employers, Grand Rapids\\' diverse economy \\ncontinue s to grow, offering endless career opportunities. es.   \\n######################################################################## To unsubscribe \\nfrom the CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 T his message was issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing list hosted by www.jiscmail.ac.uk, terms & conditions \\nare available at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print A \\npost -doctoral Research Associate  position is available immediately in our lab for structural studies of \\nhuman protein complexes important for DNA replication and repair. For more details and how to apply \\nplease visit: https://www.jobs.cam.ac.uk/job/41419/ Informal enquiries to [log in to  \\nunmask]<mailto:[log in to unmask]> are welcome. Deadline is July 14th. Best wishes, Luca Luca Pellegrini, \\nPhD Department of Biochemistry University of Cambridge Cambridge CB2 1GA UK \\n######################################################################## To unsubscribe \\nfrom the CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message was issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing list hosted by www.jiscmail.ac.uk, terms & conditions \\nare available at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print At our \\norganisation the push to do this was from the perspective of being able to accurately recharge users for \\ncluster usage.  The fundamental problem was the sam e - cryoSPARC usage needs to be reliably tied to a \\nparticular account.  There were additional restrictions because of the way our cluster is set up.  In \\nparticular, web browsers cannot normally connect to a cryoSPARC instance running on the cluster. The \\nsolution our scientific computing team adopted was to use Open OnDemand \\n(https://openondemand.org/).  Users log into an Open OnDemand web server.  This starts cryoSPARC \\nrunning in a Singularity container and launches a web based VNC session presenting a web browser with \\naccess to that container.  cryoSPARC is set up to submit processing jobs to the cluster; the container \\nrunning the cryoSPARC master stays running for the duration of the cryoSPARC processing session. The \\nare complications with this approach, i ncluding:   *   As well as a container, you need some way of \\nensuring that each user has their own cryoSPARC database that persists between sessions.  For us that \\nwas done using Open OnDemand and some clever scripting.  New users have to have this database  \\ncreated automatically for them on first use.   *   Each container will need its own cryoSPARC licence.  \\nEither you have a pool of licences and thus a maximum number of containers that can run \\nsimultaneously or each user applies for their own licence.   *   The workers need the same licence as \\nthe master.  In our case this is solved by the cluster job submission script setting this up for the worker.  \\nIf you were using a different approach you would need to find a way to handle this. I\\'m happy to give you \\nfurther information if it would be helpful.  Unfortunately, the scientific computing team member who ', \"set this up moved to another organisation last year and we lost some expertise in the process. Regards, \\nChris (Regards to Ewan as well, who is now causing t hese problems for you instead of me) -- Chris \\nRichardson | Scientific Computing - Division of Structural Biology The Institute of Cancer Research | 237 \\nFulham Road, London SW3 6JB T +442071535445 | E [log in to unmask]<mailto:[log in to unmask]> | W \\nwww.ic r.ac.uk<http://www.icr.ac.uk/> | Twitter @ICR_London<https://twitter.com/ICR_London> \\nFacebook \\nwww.facebook.com/theinstituteofcancerresearch<http://www.facebook.com/theinstituteofcancerresea\\nrch> Making the discoveries that defeat cancer [ICR Logo]<http://ww w.icr.ac.uk/> \\n________________________________ From: Collaborative Computational Project in Electron \\ncryo -Microscopy <[log in to unmask]> on behalf of Paolo Swuec <[log in to unmask]> Sent: 22 June 2023 \\n11:08 To: [log in to unmask] <[log in to unmask]> Sub ject: [ccpem] CryoSPARC to comply to ISO \\nCertificates on HPC CAUTION: This email originated from outside of the ICR. Do not click links or open \\nattachments unless you recognize the sender's email address and know the content is safe. Dear All, We \\nare havin g a hard time handling cryoSPARC on our IT infrastructure and we would love to hear your \\nfeedback and advice. In its current implementation, cryoSPARC runs under one master account \\n(cryosparc_user), which is shared by everybody logged into cryoSPARC. All f iles are therefore written and \\nmanaged by the master account, posing a serious issue with data tracking and accountability. Being a \\npublic funded institute, at Human Technopole we comply to European ISO certifications, and we need to \\nguarantee and maintain  data tracking on our IT infrastructure. The solution we have currently adopted is \\nputting the data to be processed by cryoSPARC on a segregated server, with the inconvenience of going \\nback and forth between that  quarantine area  and the main storage. Sti ll, it would be nice to easily \\nhop from Relion to CS within the same directory. We are really eager to hear from other solutions \\nadopted from any other research institute or even companies. Best, Paolo             Paolo Swuec \\nHead of Cryo EM Facility Cryo EM Facility Human Technopole Palazzo Italia Viale Rita Levi Montalcini, 1 \\n20157 Milan, Italy T. +39 02 -30247442<tel:02 -30247442> M. +39 3427758814<tel:3427758814> E. [log \\nin to unmask]<mailto:[log in to unmask]> \\n[cid:image647418.png@F56A7845.6FBA6BF2]<http s://www.facebook.com/htechnopole> \\n[cid:image609817.png@7501F075.8C77D5E5]<https://twitter.com/humantechnopole> \\n[cid:image850345.png@F89679B7.5D6C0780]<https://www.linkedin.com/company/htechnopole> \\n[cid:image641637.png@8102A211.7FD6C754]<https://www.instagr am.com/humantechnopole/> \\n[cid:image538247.jpg@89AB3B01.86D93597] Prima di stampare questa e -mail, pensa all ambiente. \\nBefore printing this e -mail, think of the environment Legal Disclaimer The information contained in or \\naccompanying this email and any att achment thereto, is intended solely for the use of the stated \\nrecipient(s) and may contain information that is confidential and/or privileged. Any dissemination, \\ndistribution or copying of this email and any attachment by anyone who is not the stated recip ient is \\nstrictly prohibited. If you receive this message in error, please notify the sender immediately and delete \\nthe message and any attachment from your system without retaining a copy. \\n________________________________ To unsubscribe from the CCPEM list , click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 The Institute of Cancer Research: \\nRoyal Cancer Hospital, a charitable Company Limited by Guarantee, Registered in England under \\nCompany No. 534147 with its Register ed Office at 123 Old Brompton Road, London SW7 3RP . This e -mail \\nmessage is confidential and for use by the addressee only. If the message is received by anyone other \\nthan the addressee, please return the message to the sender by replying to it and then del ete the \\nmessage from your computer and network. \", '######################################################################## To unsubscribe \\nfrom the CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This mess age was issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing list hosted by www.jiscmail.ac.uk, terms & conditions \\nare available at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print Dear All, \\nWe are having a hard time handli ng cryoSPARC on our IT infrastructure and we would love to hear your \\nfeedback and advice. In its current implementation, cryoSPARC runs under one master account \\n(cryosparc_user), which is shared by everybody logged into cryoSPARC. All files are therefore w ritten and \\nmanaged by the master account, posing a serious issue with data tracking and accountability. Being a \\npublic funded institute, at Human Technopole we comply to European ISO certifications, and we need to \\nguarantee and maintain data tracking on ou r IT infrastructure. The solution we have currently adopted is \\nputting the data to be processed by cryoSPARC on a segregated server, with the inconvenience of going \\nback and forth between that  quarantine area  and the main storage. Still, it would be nice  to easily \\nhop from Relion to CS within the same directory. We are really eager to hear from other solutions \\nadopted from any other research institute or even companies. Best, Paolo             Paolo Swuec \\nHead of Cryo -EM Facility Cryo -EM Facility Human Te chnopole Palazzo Italia Viale Rita Levi Montalcini, 1 \\n20157 Milan, Italy T. +39 02 -30247442 M. +39 3427758814 E. [log in to unmask] Prima di stampare \\nquesta e -mail, pensa all ambiente. Before printing this e -mail, think of the environment Legal Disclaimer  \\nThe information contained in or accompanying this email and any attachment thereto, is intended solely \\nfor the use of the stated recipient(s) and may contain information that is confidential and/or privileged. \\nAny dissemination, distribution or copying of  this email and any attachment by anyone who is not the \\nstated recipient is strictly prohibited. If you receive this message in error, please notify the sender \\nimmediately and delete the message and any attachment from your system without retaining a copy.  \\n######################################################################## To unsubscribe \\nfrom the CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message was issued to \\nmembers of www.jiscmail.ac.uk /CCPEM, a mailing list hosted by www.jiscmail.ac.uk, terms & conditions \\nare available at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print Come join \\nus in Copenhagen as a structural biology *Postdoc* to do cutting -edge research on human membrane \\ntransporters! Find more details about the position and apply here: \\nhttps://candidate.hr -manager.net/ApplicationInit.aspx/?cid=1307&departmentId=19218&ProjectId=159\\n564&MediaId=5&SkipAdvertisement=false Deadline: 12th of July Contact me at *[log in to unmask] <[log \\nin to unmask]>* if you need more information. Azadeh ---- *Azadeh Shahsavar, PhD* *University of \\nCopenhagen* Faculty of Health and Medical Sciences Department of Drug Design and Pharmacology \\nJagtvej 162 DK -2100 Copenhagen *Email: [log in to unmask] <[log in to unmask]>* \\n######################################################################## To unsubscribe \\nfrom the CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message was  issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing list hosted by www.jiscmail.ac.uk, terms & conditions \\nare available at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print Dear all, \\nWe offer a tenure track scientist posit ion in \"Structural Biology of Microtubules\" at the Paul Scherrer \\nInstitute, Switzerland. Your tasks: You will study the structure -function relationship of proteins and drugs \\nregulating microtubules using structural biology approaches, including X -ray cryst allography and \\ncryo -electron microscopy, in combination with biophysical, biochemical, and cell biology methods. Your ', 'tasks will be: - Production of high -quality recombinant protein samples using bacterial and eukaryotic \\nexpression systems - Characterizati on of protein samples using biophysical and biochemical methods - \\nCollection, analysis and interpretation of X -ray crystallography and cryo -electron microscopy data - \\nWriting of scientific publications and applications for research funding Your profile: - You have completed \\na PhD degree in structural biology, biochemistry, or molecular biology with additional post -doctoral \\neducation seen as an extra qualification - The ideal candidate is experienced in both the X -ray \\ncrystallography and cryo -electron micros copy analysis of protein complexes and is knowledgeable in the \\nfield of microtubule biology - A track record on using cell biology techniques - You enjoy working in a \\nteam of scientific staff and you are motivated to propose new ideas and discuss such with  your \\ncolleagues or collaborators Interested applicants are welcome to apply at \\nhttps://www.psi.ch/en/pa/job -opportunities/57621 -scientist -tenure -track Best wishes, Michel \\n######################################################################## To unsubscr ibe \\nfrom the CCPEM list, click the following link: \\nhttps://www.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message was issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing list hosted by www.jiscmail.ac.uk, terms & conditions \\nare available  at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print Dear all, \\nElectron Bio -Imaging Centre at Diamond Light Source Ltd has a postdoctoral position available for \\ndevelopment of novel concepts and software tools for cryo -Electron  Microscopy data analysis. If you \\nhave a PhD or equivalent in mathematics, computational sciences, physics, biophysics or other relevant \\nfields and are interested in novel software development for cryo -EM, please apply via following link \\nbefore the 9th of July (BST time): \\nhttps://vacancies.diamond.ac.uk/vacancy/pdra -in-biological -cryo -em-software -development -528794.ht\\nml eBIC is the UK national centre for cryoEM data acquisition, providing free of charge access to \\nthe-state -of-the-art instruments for the UK and international academic users. In collaboration with other \\nscientists, we strive to develop efficient on -the-fly data processing pipelines for user benefit \\n(https://www.diamond.ac.uk/Instruments/Biological -Cryo -Imaging/eBIC.html). eBIC is part of Diamon d \\nLight Source Ltd (Diamond) - the UK national synchrotron radiation source \\n(https://www.diamond.ac.uk/Home.html), located in collaboration stimulating scientific environment of \\nHarwell Science and Innovation Campus in Oxfordshire, UK \\n(https://www.harwellca mpus.com<https://www.harwellcampus.com/> ). The work on this project will \\nbe carried out in close collaboration with Diamond data analysis group, STFC Scientific Machine learning, \\nand CCPEM groups. It will complement and enhance the work undertaken as part  of the recently \\nawarded pan -European INFRAtech fragment screening and automation grant aimed at super -charging \\nstructure led drug discovery.  About You You should have a PhD degree or equivalent and have \\nexperience in algorithm design and programming for scientific data analysis. You will need to \\ndemonstrate a track record in conducting high quality original research. Knowledge and research \\nexperience in using statistical methods and machine learning for data analysis are essential. Experience \\nwith image p rocessing and cryoEM/ET data analysis is desirable but not strictly necessary. For further \\ninformation about this role please contact Yuriy Chaban ([log in to unmask]<mailto:[log in to unmask]>). \\nBenefits Diamond offers an exceptional benefits package to s upport staff in achieving a positive work/life \\nbalance. This includes 25 days annual leave plus 13 days of statutory and company holidays and 2 annual \\nvolunteering days, along with flexible working hours and an excellent pension scheme. Staff also have \\naccess to a range of amenities on site including a nursery, cafes, a restaurant and sports and leisure \\nfacilities. A relocation allowance may also be available where applicable. Best regards, Yuriy Chaban, PhD ', \"Principal EM Scientist Diamond House, Diamond Lig ht Source Ltd, Harwell Science and innovation \\nCampus, Didcot, OX11 0DE Oxfordshire, UK Tel: +44 (0) 1235 77 8207 e -mail: [log in to \\nunmask]<mailto:[log in to unmask]> --  This e -mail and any attachments may contain confidential, \\ncopyright and or privileged  material, and are for the use of the intended addressee only. If you are not \\nthe intended addressee or an authorised recipient of the addressee please notify us of receipt by \\nreturning the e -mail and do not use, copy, retain, distribute or disclose the in formation in or attached to \\nthe e -mail. Any opinions expressed within this e -mail are those of the individual and not necessarily of \\nDiamond Light Source Ltd.  Diamond Light Source Ltd. cannot guarantee that this e -mail or any \\nattachments are free from vir uses and we cannot accept liability for any damage which you may sustain \\nas a result of software viruses which may be transmitted in or with the message. Diamond Light Source \\nLimited (company no. 4375679). Registered in England and Wales with its registere d office at Diamond \\nHouse, Harwell Science and Innovation Campus, Didcot, Oxfordshire, OX11 0DE, United Kingdom \\n######################################################################## To unsubscribe \\nfrom the CCPEM list, click the following link: \\nhttps://w ww.jiscmail.ac.uk/cgi -bin/WA -JISC.exe?SUBED1=CCPEM&A=1 This message was issued to \\nmembers of www.jiscmail.ac.uk/CCPEM, a mailing list hosted by www.jiscmail.ac.uk, terms & conditions \\nare available at https://www.jiscmail.ac.uk/policyandsecurity/ JISCMail - CCPEM Archives Print Dear all, \\nThe Bertini Award 2024<https://instruct -eric.eu/tracker/nSkCRz> call is open, submit your nominations \\nfor the winner before the deadline of 30 September 2023. Nominations are invited for the Award for a \\nsignificant achieveme nt in any area of integrated structural biology. Self -nominations are not accepted. \\nIvano Bertini was a pioneer of innovative methodologies in the field of NMR spectroscopy who \\nchampioned technology integration as a foundation for correlative structural bi ology. Instruct -ERIC \\npresents an award in his name at the Biennial Structural Biology \\nConference<https://instruct -eric.eu/tracker/nGYgq7>, recognising scientists who have undertaken \\nfrontier research utilising an integrative structural biology approach whi ch aligns with the Instruct vision. \\nThe award will be given solely on the basis of excellence in research and must include structural data as a \\nkey component of the research achievement. The award recognises a clearly defined breakthrough \\n(either in scient ific knowledge or technology development) and is not for a lifetime achievement. Submit \\nyour nominations here.<https://instruct -eric.eu/tracker/nSkCRz> Take a look at the winner of last year's \\naward, Sjors Scheres, here.<https://instruct -eric.eu/tracker/zm DxPa> \\n########################################################################  \"]\n"
     ]
    }
   ],
   "source": [
    "from PyPDF2 import PdfReader\n",
    "\n",
    "def load_pdf_to_array(file_path):\n",
    "    # Open the file in read-binary mode\n",
    "    with open(file_path, 'rb') as file:\n",
    "        # Initialize a PDF file reader object\n",
    "        pdf_reader = PdfReader(file)\n",
    "        \n",
    "        # Check if the PDF is encrypted\n",
    "        if pdf_reader.is_encrypted:\n",
    "            print(\"PDF is encrypted. Unable to read.\")\n",
    "            return None\n",
    "        \n",
    "        # Get the number of pages in the PDF\n",
    "        num_pages = len(pdf_reader.pages)\n",
    "\n",
    "        # Initialize an empty list to hold the pages\n",
    "        page_list = []\n",
    "\n",
    "        # Loop through each page in the PDF\n",
    "        for page in pdf_reader.pages:\n",
    "            # Get the text of the page\n",
    "            text = page.extract_text()\n",
    "            # Add the text to the list\n",
    "            page_list.append(text)\n",
    "\n",
    "    return page_list\n",
    "\n",
    "# Use the function\n",
    "pdf_array = load_pdf_to_array('C:/crawl/temptestingpdf.pdf')\n",
    "print(pdf_array)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "59033be5",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Now you have 96 documents,done\n"
     ]
    }
   ],
   "source": [
    "from langchain.document_loaders import DirectoryLoader\n",
    "from langchain.text_splitter import CharacterTextSplitter\n",
    "\n",
    "loader =  DirectoryLoader(r'C:\\crawl\\testing')\n",
    "docs = loader.load()\n",
    "char_text = CharacterTextSplitter(chunk_size=2000, chunk_overlap=0)\n",
    "doc_texts = char_text.split_documents(docs)\n",
    "print (f'Now you have {len(doc_texts)} documents,done')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "e4d637c3",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Now you have 96 documents\n"
     ]
    }
   ],
   "source": [
    "# print(doc_texts)\n",
    "print (f'Now you have {len(doc_texts)} documents')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "590b5a73",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\Users\\12146\\miniconda3\\lib\\site-packages\\pinecone\\index.py:4: TqdmWarning: IProgress not found. Please update jupyter and ipywidgets. See https://ipywidgets.readthedocs.io/en/stable/user_install.html\n",
      "  from tqdm.autonotebook import tqdm\n"
     ]
    }
   ],
   "source": [
    "from langchain.vectorstores import Chroma,Pinecone\n",
    "from langchain.embeddings.openai import OpenAIEmbeddings\n",
    "import pinecone"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "fe5afc21",
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "# Check to see if there is an environment variable with you API keys, if not, use what you put below\n",
    "OPENAI_API_KEY = os.environ.get('OPENAI_API_KEY', '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')\n",
    "\n",
    "PINECONE_API_KEY = os.environ.get('PINECONE_API_KEY', 'bacc7a5f-0b08-4aae-8695-6fa9e0f458a9')\n",
    "PINECONE_API_ENV = os.environ.get('PINECONE_API_ENV', 'us-west1-gcp-free') # You may need to switch with your en"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "78401894",
   "metadata": {},
   "outputs": [],
   "source": [
    "# initialize pinecone\n",
    "pinecone.init(\n",
    "    api_key=PINECONE_API_KEY,  # find at app.pinecone.io\n",
    "    environment=PINECONE_API_ENV  # next to api key in console\n",
    ")\n",
    "index_name = \"cryohub\" # put in the name of your pinecone index here"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "7210bf5e",
   "metadata": {},
   "outputs": [],
   "source": [
    "embeddings = OpenAIEmbeddings(openai_api_key=OPENAI_API_KEY)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "2687d8be",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<langchain.vectorstores.pinecone.Pinecone object at 0x000001EA6638FD60>\n"
     ]
    }
   ],
   "source": [
    "docsearch = Pinecone.from_texts([t.page_content for t in doc_texts], embeddings, index_name=index_name)\n",
    "#docsearch = <langchain.vectorstores.pinecone.Pinecone object at 0x000001EA6638FD60>\n",
    "print(docsearch)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "529cf1ab",
   "metadata": {},
   "outputs": [],
   "source": [
    "query = \"What are examples of good data science teams?\"\n",
    "docs = docsearch.similarity_search(query)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "5d020dbf",
   "metadata": {},
   "outputs": [],
   "source": [
    "from langchain.llms import OpenAI\n",
    "from langchain.chains.question_answering import load_qa_chain"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "ca7dfd6d",
   "metadata": {},
   "outputs": [],
   "source": [
    "llm = OpenAI(temperature=0, openai_api_key=OPENAI_API_KEY)\n",
    "chain = load_qa_chain(llm, chain_type=\"stuff\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "30331c81",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "' Examples of good data science teams include teams that are able to mine large datasets, build machine learning-based predictive models, dock virtual compound databases, include quantum chemistry calculations in chemical synthesis planning, run molecular simulations of large protein complexes, or use deep learning models for cell segmentation image recognition. Additionally, good data science teams should have a collaborative culture that allows teams of people from multiple disciplines to work together to achieve more than any single person can achieve alone.'"
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "chain.run(input_documents=docs, question=query)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "id": "4155ea95",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'dimension': 1536,\n",
       " 'index_fullness': 0.1,\n",
       " 'namespaces': {'': {'vector_count': 11616}},\n",
       " 'total_vector_count': 11616}"
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "index = pinecone.Index(index_name)\n",
    "index.describe_index_stats()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "id": "69577d5f",
   "metadata": {},
   "outputs": [
    {
     "ename": "AttributeError",
     "evalue": "module 'pinecone' has no attribute 'query'",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mAttributeError\u001b[0m                            Traceback (most recent call last)",
      "Cell \u001b[1;32mIn[21], line 8\u001b[0m\n\u001b[0;32m      5\u001b[0m query_vector \u001b[38;5;241m=\u001b[39m [\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mwhat is cryohub?\u001b[39m\u001b[38;5;124m\"\u001b[39m]  \u001b[38;5;66;03m# replace this with your actual query vector\u001b[39;00m\n\u001b[0;32m      7\u001b[0m \u001b[38;5;66;03m# Query Pinecone for the nearest neighbors of a given vector\u001b[39;00m\n\u001b[1;32m----> 8\u001b[0m results \u001b[38;5;241m=\u001b[39m \u001b[43mpinecone\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mquery\u001b[49m(index_name\u001b[38;5;241m=\u001b[39mindex_name, queries\u001b[38;5;241m=\u001b[39m[query_vector], top_k\u001b[38;5;241m=\u001b[39m\u001b[38;5;241m5\u001b[39m)\n\u001b[0;32m     10\u001b[0m \u001b[38;5;28mprint\u001b[39m(results)\n",
      "\u001b[1;31mAttributeError\u001b[0m: module 'pinecone' has no attribute 'query'"
     ]
    }
   ],
   "source": [
    "# Name of your index\n",
    "index_name = \"cryohub\"\n",
    "\n",
    "# Your query vector\n",
    "query_vector = [\"what is cryohub?\"]  # replace this with your actual query vector\n",
    "\n",
    "# Query Pinecone for the nearest neighbors of a given vector\n",
    "results = pinecone.query(index_name=index_name, queries=[query_vector], top_k=5)\n",
    "\n",
    "print(results)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "id": "ad787d20",
   "metadata": {},
   "outputs": [
    {
     "ename": "SyntaxError",
     "evalue": "invalid syntax. Perhaps you forgot a comma? (4040927678.py, line 2)",
     "output_type": "error",
     "traceback": [
      "\u001b[1;36m  Cell \u001b[1;32mIn[22], line 2\u001b[1;36m\u001b[0m\n\u001b[1;33m    vector = [\"what's cryoEM?\"]\u001b[0m\n\u001b[1;37m             ^\u001b[0m\n\u001b[1;31mSyntaxError\u001b[0m\u001b[1;31m:\u001b[0m invalid syntax. Perhaps you forgot a comma?\n"
     ]
    }
   ],
   "source": [
    "index.query(\n",
    "vector = [\"what's cryoEM?\"]\n",
    "    top_k = 3,\n",
    "    include_values=True\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "id": "10a0fbf6",
   "metadata": {},
   "outputs": [],
   "source": [
    "import pinecone\n",
    "\n",
    "pinecone.init(api_key=\"bacc7a5f-0b08-4aae-8695-6fa9e0f458a9\", environment=\"us-west1-gcp-free\")\n",
    "\n",
    "index_description = pinecone.describe_index(\"cryohub\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "id": "7a906b09",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "IndexDescription(name='cryohub', metric='cosine', replicas=1, dimension=1536.0, shards=1, pods=1, pod_type='p1', status={'ready': True, 'state': 'Ready'}, metadata_config=None, source_collection='')\n"
     ]
    }
   ],
   "source": [
    "print(index_description)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "91c72f97",
   "metadata": {},
   "outputs": [],
   "source": [
    "#active_indexes = pinecone.list_indexes()\n",
    "#docsearch = langchain.vectorstores.pinecone.Pinecone object at 0x000001EA6638FD60\n",
    "# import langchain.vectorstores.pinecone\n",
    "# docsearch = langchain.vectorstores.pinecone.Pinecone(0x000001EA6638FD60)\n",
    "docsearch = Pinecone.from_existing_index(index_name, embeddings)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "832cb228",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<langchain.vectorstores.pinecone.Pinecone object at 0x000001943CA6D000>\n"
     ]
    }
   ],
   "source": [
    "print(docsearch)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "6c9f228d",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Note: you may need to restart the kernel to use updated packages.\n"
     ]
    }
   ],
   "source": [
    "pip freeze > requirements.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "8c1e7a29",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.10"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
